<!DOCTYPE html>
<!-- START: inst/pkgdown/templates/layout.html --><!-- Generated by pkgdown: do not edit by hand --><html lang="en" data-bs-theme="auto">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<title>Introduction to Genome Browsers: All in One View</title>
<meta name="viewport" content="width=device-width, initial-scale=1">
<script src="../assets/themetoggle.js"></script><link rel="stylesheet" type="text/css" href="../assets/styles.css">
<script src="../assets/scripts.js" type="text/javascript"></script><!-- mathjax --><script type="text/x-mathjax-config">
    MathJax.Hub.Config({
      config: ["MMLorHTML.js"],
      jax: ["input/TeX","input/MathML","output/HTML-CSS","output/NativeMML", "output/PreviewHTML"],
      extensions: ["tex2jax.js","mml2jax.js","MathMenu.js","MathZoom.js", "fast-preview.js", "AssistiveMML.js", "a11y/accessibility-menu.js"],
      TeX: {
        extensions: ["AMSmath.js","AMSsymbols.js","noErrors.js","noUndefined.js"]
      },
      tex2jax: {
        inlineMath: [['\\(', '\\)']],
        displayMath: [ ['$$','$$'], ['\\[', '\\]'] ],
        processEscapes: true
      }
    });
    </script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><!-- Responsive Favicon for The Carpentries --><link rel="apple-touch-icon" sizes="180x180" href="../favicons/uom/apple-touch-icon.png">
<link rel="icon" type="image/png" sizes="32x32" href="../favicons/uom/favicon-32x32.png">
<link rel="icon" type="image/png" sizes="16x16" href="../favicons/uom/favicon-16x16.png">
<link rel="manifest" href="../favicons/uom/site.webmanifest">
<link rel="mask-icon" href="../favicons/uom/safari-pinned-tab.svg" color="#5bbad5">
<meta name="msapplication-TileColor" content="#da532c">
<meta name="theme-color" media="(prefers-color-scheme: light)" content="white">
<meta name="theme-color" media="(prefers-color-scheme: dark)" content="black">
</head>
<body>
    <header id="top" class="navbar navbar-expand-md top-nav uom"><svg xmlns="http://www.w3.org/2000/svg" class="d-none"><symbol id="check2" viewbox="0 0 16 16"><path d="M13.854 3.646a.5.5 0 0 1 0 .708l-7 7a.5.5 0 0 1-.708 0l-3.5-3.5a.5.5 0 1 1 .708-.708L6.5 10.293l6.646-6.647a.5.5 0 0 1 .708 0z"></path></symbol><symbol id="circle-half" viewbox="0 0 16 16"><path d="M8 15A7 7 0 1 0 8 1v14zm0 1A8 8 0 1 1 8 0a8 8 0 0 1 0 16z"></path></symbol><symbol id="moon-stars-fill" viewbox="0 0 16 16"><path d="M6 .278a.768.768 0 0 1 .08.858 7.208 7.208 0 0 0-.878 3.46c0 4.021 3.278 7.277 7.318 7.277.527 0 1.04-.055 1.533-.16a.787.787 0 0 1 .81.316.733.733 0 0 1-.031.893A8.349 8.349 0 0 1 8.344 16C3.734 16 0 12.286 0 7.71 0 4.266 2.114 1.312 5.124.06A.752.752 0 0 1 6 .278z"></path><path d="M10.794 3.148a.217.217 0 0 1 .412 0l.387 1.162c.173.518.579.924 1.097 1.097l1.162.387a.217.217 0 0 1 0 .412l-1.162.387a1.734 1.734 0 0 0-1.097 1.097l-.387 1.162a.217.217 0 0 1-.412 0l-.387-1.162A1.734 1.734 0 0 0 9.31 6.593l-1.162-.387a.217.217 0 0 1 0-.412l1.162-.387a1.734 1.734 0 0 0 1.097-1.097l.387-1.162zM13.863.099a.145.145 0 0 1 .274 0l.258.774c.115.346.386.617.732.732l.774.258a.145.145 0 0 1 0 .274l-.774.258a1.156 1.156 0 0 0-.732.732l-.258.774a.145.145 0 0 1-.274 0l-.258-.774a1.156 1.156 0 0 0-.732-.732l-.774-.258a.145.145 0 0 1 0-.274l.774-.258c.346-.115.617-.386.732-.732L13.863.1z"></path></symbol><symbol id="sun-fill" viewbox="0 0 16 16"><path d="M8 12a4 4 0 1 0 0-8 4 4 0 0 0 0 8zM8 0a.5.5 0 0 1 .5.5v2a.5.5 0 0 1-1 0v-2A.5.5 0 0 1 8 0zm0 13a.5.5 0 0 1 .5.5v2a.5.5 0 0 1-1 0v-2A.5.5 0 0 1 8 13zm8-5a.5.5 0 0 1-.5.5h-2a.5.5 0 0 1 0-1h2a.5.5 0 0 1 .5.5zM3 8a.5.5 0 0 1-.5.5h-2a.5.5 0 0 1 0-1h2A.5.5 0 0 1 3 8zm10.657-5.657a.5.5 0 0 1 0 .707l-1.414 1.415a.5.5 0 1 1-.707-.708l1.414-1.414a.5.5 0 0 1 .707 0zm-9.193 9.193a.5.5 0 0 1 0 .707L3.05 13.657a.5.5 0 0 1-.707-.707l1.414-1.414a.5.5 0 0 1 .707 0zm9.193 2.121a.5.5 0 0 1-.707 0l-1.414-1.414a.5.5 0 0 1 .707-.707l1.414 1.414a.5.5 0 0 1 0 .707zM4.464 4.465a.5.5 0 0 1-.707 0L2.343 3.05a.5.5 0 1 1 .707-.707l1.414 1.414a.5.5 0 0 1 0 .708z"></path></symbol></svg><a class="visually-hidden-focusable skip-link" href="#main-content">Skip to main content</a>
  <div class="container-fluid top-nav-container">
    <div class="col-md-8">
      <div class="large-logo">
        <img id="uom-logo" alt="This tutorial will introduce you to the genome browser format and illustrate how some freely available genome browsers can be used to interrogate a variety of data types, such as gene expression, genomic variation, methylation and many more." src="../assets/images/uom-logo.svg"><span class="badge text-bg-danger">
          <abbr title="This lesson is in the pre-alpha phase, which means that it is in early development, but has not yet been taught.">
            <a href="https://docs.carpentries.org/resources/curriculum/lesson-life-cycle.html" class="external-link alert-link">
              <i aria-hidden="true" class="icon" data-feather="alert-octagon" style="border-radius: 5px"></i>
              Pre-Alpha
            </a>
            <span class="visually-hidden">This lesson is in the pre-alpha phase, which means that it is in early development, but has not yet been taught.</span>
          </abbr>
        </span>

      </div>
    </div>
    <div class="selector-container">
      <div id="theme-selector">
        <li class="nav-item dropdown" id="theme-button-list">
          <button class="btn btn-link nav-link px-0 px-lg-2 dropdown-toggle d-flex align-items-center" id="bd-theme" type="button" aria-expanded="false" data-bs-toggle="dropdown" data-bs-display="static" aria-label="Toggle theme (auto)">
            <svg class="bi my-1 theme-icon-active"><use href="#circle-half"></use></svg><i data-feather="chevron-down"></i>
          </button>
          <ul class="dropdown-menu dropdown-menu-end" aria-labelledby="bd-theme-text">
<li>
              <button type="button" class="btn dropdown-item d-flex align-items-center" data-bs-theme-value="light" aria-pressed="false">
                <svg class="bi me-2 theme-icon"><use href="#sun-fill"></use></svg>
                Light
                <svg class="bi ms-auto d-none"><use href="#check2"></use></svg></button>
            </li>
            <li>
              <button type="button" class="btn dropdown-item d-flex align-items-center" data-bs-theme-value="dark" aria-pressed="false">
                <svg class="bi me-2 theme-icon"><use href="#moon-stars-fill"></use></svg>
                Dark
                <svg class="bi ms-auto d-none"><use href="#check2"></use></svg></button>
            </li>
            <li>
              <button type="button" class="btn dropdown-item d-flex align-items-center active" data-bs-theme-value="auto" aria-pressed="true">
                <svg class="bi me-2 theme-icon"><use href="#circle-half"></use></svg>
                Auto
                <svg class="bi ms-auto d-none"><use href="#check2"></use></svg></button>
            </li>
          </ul>
</li>
      </div>

      <div class="dropdown" id="instructor-dropdown">
        <button class="btn btn-secondary dropdown-toggle bordered-button" type="button" id="dropdownMenu1" data-bs-toggle="dropdown" aria-expanded="false">
          <i aria-hidden="true" class="icon" data-feather="eye"></i> Instructor View <i data-feather="chevron-down"></i>
        </button>
        <ul class="dropdown-menu" aria-labelledby="dropdownMenu1">
<li><button class="dropdown-item" type="button" onclick="window.location.href='../aio.html';">Learner View</button></li>
        </ul>
</div>
    </div>
  </div>
  <hr></header><nav class="navbar navbar-expand-xl bottom-nav uom" aria-label="Main Navigation"><div class="container-fluid nav-container">
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle Navigation">
      <span class="navbar-toggler-icon"></span>
      <span class="menu-title">Menu</span>
    </button>
    <div class="nav-logo">
      <img class="small-logo" alt="This tutorial will introduce you to the genome browser format and illustrate how some freely available genome browsers can be used to interrogate a variety of data types, such as gene expression, genomic variation, methylation and many more." src="../assets/images/uom-logo-sm.svg">
</div>
    <div class="lesson-title-md">
      Introduction to Genome Browsers
    </div>
    <div class="search-icon-sm">
      <!-- TODO: do not show until we have search
        <i role="img" aria-label="Search the All In One page" data-feather="search"></i>
      -->
    </div>
    <div class="desktop-nav">
      <ul class="navbar-nav me-auto mb-2 mb-lg-0">
<li class="nav-item">
          <span class="lesson-title">
            Introduction to Genome Browsers
          </span>
        </li>
        <li class="nav-item">
          <a class="nav-link" href="../instructor/key-points.html">Key Points</a>
        </li>
        <li class="nav-item">
          <a class="nav-link" href="../instructor/instructor-notes.html">Instructor Notes</a>
        </li>
        <li class="nav-item">
          <a class="nav-link" href="../instructor/images.html">Extract All Images</a>
        </li>
        <li class="nav-item dropdown">
          <button class="nav-link dropdown-toggle" id="navbarDropdown" data-bs-toggle="dropdown" aria-expanded="false">
            More <i data-feather="chevron-down"></i>
          </button>
          <ul class="dropdown-menu" aria-labelledby="navbarDropdown">
<hr>
<li><a class="dropdown-item" href="reference.html">Reference</a></li>
          </ul>
</li>
      </ul>
</div>
    <!--
    <form class="d-flex col-md-2 search-form">
      <fieldset disabled>
      <input class="form-control me-2 searchbox" type="search" placeholder="" aria-label="">
        <button class="btn btn-outline-success tablet-search-button"  type="submit">
          <i class="search-icon" data-feather="search" role="img" aria-label="Search the All In One page"></i>
        </button>
      </fieldset>
    </form>
    -->
    <a id="search-button" class="btn btn-primary" href="../instructor/aio.html" role="button" aria-label="Search the All In One page">Search the All In One page</a>
  </div>
<!--/div.container-fluid -->
</nav><div class="col-md-12 mobile-title">
  Introduction to Genome Browsers
</div>

<aside class="col-md-12 lesson-progress"><div style="width: %" class="percentage">
    %
  </div>
  <div class="progress uom">
    <div class="progress-bar uom" role="progressbar" style="width: %" aria-valuenow="" aria-label="Lesson Progress" aria-valuemin="0" aria-valuemax="100">
    </div>
  </div>
</aside><div class="container">
      <div class="row">
        <!-- START: inst/pkgdown/templates/navbar.html -->
<div id="sidebar-col" class="col-lg-4">
  <div id="sidebar" class="sidebar">
      <nav aria-labelledby="flush-headingEleven"><button role="button" aria-label="close menu" alt="close menu" aria-expanded="true" aria-controls="sidebar" class="collapse-toggle" data-collapse="Collapse " data-episodes="Episodes ">
          <i class="search-icon" data-feather="x" role="img"></i>
        </button>
        <div class="sidebar-inner">
          <div class="row mobile-row" id="theme-row-mobile">
            <div class="col" id="theme-selector">
              <li class="nav-item dropdown" id="theme-button-list">
                <button class="btn btn-link nav-link px-0 px-lg-2 dropdown-toggle d-flex align-items-center" id="bd-theme" type="button" aria-expanded="false" data-bs-toggle="dropdown" data-bs-display="static" aria-label="Toggle theme (auto)">
                  <svg class="bi my-1 theme-icon-active"><use href="#circle-half"></use></svg><span class="d-lg-none ms-1" id="bd-theme-text">Toggle Theme</span>
                </button>
                <ul class="dropdown-menu dropdown-menu-right" aria-labelledby="bd-theme-text">
<li>
                    <button type="button" class="btn dropdown-item d-flex align-items-center" data-bs-theme-value="light" aria-pressed="false">
                      <svg class="bi me-2 theme-icon"><use href="#sun-fill"></use></svg>
                      Light
                      <svg class="bi ms-auto d-none"><use href="#check2"></use></svg></button>
                  </li>
                  <li>
                    <button type="button" class="btn dropdown-item d-flex align-items-center" data-bs-theme-value="dark" aria-pressed="false">
                      <svg class="bi me-2 theme-icon"><use href="#moon-stars-fill"></use></svg>
                      Dark
                      <svg class="bi ms-auto d-none"><use href="#check2"></use></svg></button>
                  </li>
                  <li>
                    <button type="button" class="btn dropdown-item d-flex align-items-center active" data-bs-theme-value="auto" aria-pressed="true">
                      <svg class="bi me-2 theme-icon"><use href="#circle-half"></use></svg>
                      Auto
                      <svg class="bi ms-auto d-none"><use href="#check2"></use></svg></button>
                  </li>
                </ul>
</li>
            </div>
          </div>
          <div class="row mobile-row">
            <div class="col">
              <div class="sidenav-view-selector">
                <div class="accordion accordion-flush" id="accordionFlush9">
                  <div class="accordion-item">
                    <h2 class="accordion-header" id="flush-headingNine">
                      <button class="accordion-button collapsed" id="instructor" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapseNine" aria-expanded="false" aria-controls="flush-collapseNine">
                        <i id="eye" aria-hidden="true" class="icon" data-feather="eye"></i> Instructor View
                      </button>
                    </h2>
                    <div id="flush-collapseNine" class="accordion-collapse collapse" aria-labelledby="flush-headingNine" data-bs-parent="#accordionFlush2">
                      <div class="accordion-body">
                        <a href="../aio.html">Learner View</a>
                      </div>
                    </div>
                  </div>
<!--/div.accordion-item-->
                </div>
<!--/div.accordion-flush-->
              </div>
<!--div.sidenav-view-selector -->
            </div>
<!--/div.col -->

            <hr>
</div>
<!--/div.mobile-row -->

          <div class="accordion accordion-flush" id="accordionFlush11">
            <div class="accordion-item">

              <button id="chapters" class="accordion-button show" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapseEleven" aria-expanded="false" aria-controls="flush-collapseEleven">
                <h2 class="accordion-header chapters" id="flush-headingEleven">
                  EPISODES
                </h2>
              </button>
              <div id="flush-collapseEleven" class="accordion-collapse show collapse" aria-labelledby="flush-headingEleven" data-bs-parent="#accordionFlush11">

                <div class="accordion-body">
                  <div class="accordion accordion-flush" id="accordionFlush1">
  <div class="accordion-item">
    <div class="accordion-header" id="flush-heading1">
        <a href="index.html">Summary and Schedule</a>
    </div>
<!--/div.accordion-header-->

  </div>
<!--/div.accordion-item-->
</div>
<!--/div.accordion-flush-->

<div class="accordion accordion-flush" id="accordionFlush2">
  <div class="accordion-item">
    <div class="accordion-header" id="flush-heading2">
        <a href="01_genome_browser_background.html">1. Genome browsers</a>
    </div>
<!--/div.accordion-header-->

  </div>
<!--/div.accordion-item-->
</div>
<!--/div.accordion-flush-->

<div class="accordion accordion-flush" id="accordionFlush3">
  <div class="accordion-item">
    <div class="accordion-header" id="flush-heading3">
        <a href="02_data_background.html">2. Data used in this workshop</a>
    </div>
<!--/div.accordion-header-->

  </div>
<!--/div.accordion-item-->
</div>
<!--/div.accordion-flush-->

<div class="accordion accordion-flush" id="accordionFlush4">
  <div class="accordion-item">
    <div class="accordion-header" id="flush-heading4">
        <a href="03_UCSC_setup.html">3. UCSC Genome Browser: Setup</a>
    </div>
<!--/div.accordion-header-->

  </div>
<!--/div.accordion-item-->
</div>
<!--/div.accordion-flush-->

<div class="accordion accordion-flush" id="accordionFlush5">
  <div class="accordion-item">
    <div class="accordion-header" id="flush-heading5">
        <a href="04_UCSC_GeneModels.html">4. UCSC Genome Browser: Understanding gene models</a>
    </div>
<!--/div.accordion-header-->

  </div>
<!--/div.accordion-item-->
</div>
<!--/div.accordion-flush-->

<div class="accordion accordion-flush" id="accordionFlush6">
  <div class="accordion-item">
    <div class="accordion-header" id="flush-heading6">
        <a href="05_BLAT.html">5. UCSC Genome Browser: BLAT tool</a>
    </div>
<!--/div.accordion-header-->

  </div>
<!--/div.accordion-item-->
</div>
<!--/div.accordion-flush-->

<div class="accordion accordion-flush" id="accordionFlush7">
  <div class="accordion-item">
    <div class="accordion-header" id="flush-heading7">
        <a href="06_gene_expression_data.html">6. UCSC Genome Browser: Gene expression data</a>
    </div>
<!--/div.accordion-header-->

  </div>
<!--/div.accordion-item-->
</div>
<!--/div.accordion-flush-->

<div class="accordion accordion-flush" id="accordionFlush8">
  <div class="accordion-item">
    <div class="accordion-header" id="flush-heading8">
        <a href="07_IGV.html">7. IGV</a>
    </div>
<!--/div.accordion-header-->

  </div>
<!--/div.accordion-item-->
</div>
<!--/div.accordion-flush-->

                </div>
              </div>
            </div>

            <hr class="half-width">
<div class="accordion accordion-flush lesson-resources" id="accordionFlush12">
              <div class="accordion-item">
                <h2 class="accordion-header" id="flush-headingTwelve">
                  <button class="accordion-button collapsed" id="lesson-resources" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapseTwelve" aria-expanded="false" aria-controls="flush-collapseTwelve">
                    RESOURCES
                  </button>
                </h2>
                <div id="flush-collapseTwelve" class="accordion-collapse collapse" aria-labelledby="flush-headingTwelve" data-bs-parent="#accordionFlush12">
                  <div class="accordion-body">
                    <ul>
<li>
                        <a href="../instructor/key-points.html">Key Points</a>
                      </li>
                      <li>
                        <a href="../instructor/instructor-notes.html">Instructor Notes</a>
                      </li>
                      <li>
                        <a href="../instructor/images.html">Extract All Images</a>
                      </li>
                      <hr>
<li><a class="dropdown-item" href="reference.html">Reference</a></li>
                    </ul>
</div>
                </div>
              </div>
            </div>
            <hr class="half-width lesson-resources">
<a href="../instructor/aio.html">See all in one page</a>


            <hr class="d-none d-sm-block d-md-none">
<div class="d-grid gap-1">

            </div>
          </div>
<!-- /div.accordion -->
        </div>
<!-- /div.sidebar-inner -->
      </nav>
</div>
<!-- /div.sidebar -->
  </div>
<!-- /div.sidebar-col -->
<!-- END:   inst/pkgdown/templates/navbar.html-->

        <!-- START: inst/pkgdown/templates/content-extra.html -->
  <div class="col-xl-8 col-lg-12 primary-content">
    <main id="main-content" class="main-content"><div class="container lesson-content">
        
        
<section id="aio-01_genome_browser_background"><p>Content from <a href="01_genome_browser_background.html">Genome browsers</a></p>
<hr>
<p>Last updated on 2025-07-24 |

        <a href="https://github.com/julia-broadbent/intro-to-genome-browsers/edit/main/episodes/01_genome_browser_background.md" class="external-link">Edit this page <i aria-hidden="true" data-feather="edit"></i></a></p>
<p>Estimated time: <i aria-hidden="true" data-feather="clock"></i> 10 minutes</p>
<div class="text-end">
          <button role="button" aria-pressed="false" tabindex="0" id="expand-code" class="pull-right" data-expand="Expand All Solutions " data-collapse="Collapse All Solutions "> Expand All Solutions <i aria-hidden="true" data-feather="plus"></i></button>
        </div>
<div class="overview card">
<h2 class="card-header">Overview</h2>
<div class="row g-0">
<div class="col-md-4">
<div class="card-body">
<div class="inner">
<h3 class="card-title">Questions</h3>
<ul>
<li>What are genome browsers and why are they useful?</li>
<li>How was the human reference genome constructed?</li>
</ul>
</div>
</div>
</div>
<div class="col-md-8">
<div class="card-body">
<div class="inner bordered">
<h3 class="card-title">Objectives</h3>
<ul>
<li>Understand the utility of genome browsers</li>
<li>Describe the human reference genome</li>
</ul>
</div>
</div>
</div>
</div>
</div>
<section><h2 class="section-heading" id="introduction-to-genome-browsers">Introduction to genome browsers<a class="anchor" aria-label="anchor" href="#introduction-to-genome-browsers"></a>
</h2>
<hr class="half-width">
<p>Genome browsers are invaluable for <strong>viewing and interpreting
the many different types of data that can be anchored to genomic
positions.</strong> These include variation, transcription, the many
types regulatory data such as methylation and transcription factor
binding, and disease associations. The larger genome browsers serve as
data archives for valuable public datasets facilitating
<strong>visualisation and analysis of different data types.</strong> It
is also possible to load your own data into some of the public genome
browsers.</p>
<p>By enabling viewing of one type of data in the context of another,
the use of Genome browsers can reveal important information about gene
regulation in both normal development and disease, and assist hypothesis
development relating to genotype phenotype relationships.</p>
<div id="commonly-used-genome-browsers" class="callout">
<div class="callout-square">
<i class="callout-icon" data-feather="bell"></i>
</div>
<div id="commonly-used-genome-browsers" class="callout-inner">
<h3 class="callout-title">COMMONLY USED GENOME BROWSERS</h3>
<div class="callout-content">
<p>All researchers are encouraged to become familiar with the use of
some of the main browsers, such as:</p>
<ul>
<li>
<a href="https://genome.ucsc.edu/" class="external-link">The UCSC Genome Browser</a>
(RRID:SCR_005780)</li>
<li>
<a href="https://www.ensembl.org/index.html" class="external-link">ESEMBL Genome
Browser</a> (RRID:SCR_013367)</li>
<li>
<a href="https://epigenomegateway.wustl.edu/browser/" class="external-link">Epigenome
browser at WashU</a> (RRID:SCR_006208)</li>
<li>
<a href="http://software.broadinstitute.org/software/igv/" class="external-link">Integrative
Genomics Viewer (IGV)</a> (RRID:SCR_011793)</li>
</ul>
</div>
</div>
</div>
<p>These browsers are designed for use by researchers without
programming experience and the developers often provide extensive
tutorials and cases studies demonstrating the myriad of ways in which
data can be loaded and interpreted to assist in develop and supporting
your research hypothesis.</p>
<p>Many large genomic projects also incorporate genome browsers into
their web portals to enable users to easily search and view the data.
These include:</p>
<ul>
<li><a href="https://gtexportal.org/home/" class="external-link">GTEx</a></li>
<li><a href="https://gnomad.broadinstitute.org/" class="external-link">gnomAD</a></li>
</ul></section><section><h2 class="section-heading" id="the-human-reference-genome">The human reference genome<a class="anchor" aria-label="anchor" href="#the-human-reference-genome"></a>
</h2>
<hr class="half-width">
<p>Genome browsers rely on a <strong>common reference genome</strong>
for each species in order to map data from different sources to the
correct location. A consortium has agreed on a common numbering for each
position on the genome for each species. However, this position will
vary based on the version of the genome, as error correction and updates
can change the numbering. Therefore it is very important to know
<strong>which version of the genome your data of interest is aligned
to.</strong></p>
<p>The sequence for the human reference genome was accumulated up over
many years from sequence data from many different sources and does not
represent the sequence of one single person. Instead it is a composite
of fragments of the genome from many different people. Also, unlike the
human genome which is diploid, the human reference genome is haploid.
That is there is only one copy of each chromosome. It therefore does not
reflect the variation on the population, or even the most common
variants in the human genome. <em>Exploring variation within human
genome is </em> <em>very important and facilitated by genome browsers
but not covered in this workshop.</em></p>
<div id="accordionSpoiler1" class="accordion spoiler-accordion accordion-flush">
<div class="accordion-item">
<button class="accordion-button spoiler-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapseSpoiler1" aria-expanded="false" aria-controls="collapseSpoiler1">
  <h3 class="accordion-header" id="headingSpoiler1">
  <div class="note-square"><i aria-hidden="true" class="callout-icon" data-feather="eye"></i></div>
  FURTHER READING
  </h3>
</button>
<div id="collapseSpoiler1" class="accordion-collapse collapse" aria-labelledby="headingSpoiler1" data-bs-parent="#accordionSpoiler1">
<div class="accordion-body">
<p>If you would like to learn more about genome build version number and
updates, please go to:</p>
<ul>
<li><a href="https://www.ncbi.nlm.nih.gov/grc" class="external-link">The Genome reference
consortium</a></li>
<li><a href="https://genome.ucsc.edu/FAQ/FAQreleases.html" class="external-link">UCSC genome
browser: What does the nomenclature mean?</a></li>
<li><a href="https://genome.ucsc.edu/goldenPath/newsarch.html" class="external-link">UCSC
genome browser: Updates and blog pages</a></li>
</ul>
</div>
</div>
</div>
</div>
<div id="keypoints1" class="callout keypoints">
<div class="callout-square">
<i class="callout-icon" data-feather="key"></i>
</div>
<div class="callout-inner">
<h3 class="callout-title">Key Points</h3>
<div class="callout-content">
<ul>
<li>Genome browsers rely on a common reference genome for each species
to map data to the correct location</li>
<li>Genome browsers allow us to interpret and view many different types
of data associated with genomic positions</li>
<li>Genome browsers are designed to be used by researchers without
programming experience</li>
</ul>
</div>
</div>
</div>
<!--
Place links that you need to refer to multiple times across pages here. Delete
any links that you are not going to use.
 -->
</section></section><section id="aio-02_data_background"><p>Content from <a href="02_data_background.html">Data used in this workshop</a></p>
<hr>
<p>Last updated on 2025-07-24 |

        <a href="https://github.com/julia-broadbent/intro-to-genome-browsers/edit/main/episodes/02_data_background.md" class="external-link">Edit this page <i aria-hidden="true" data-feather="edit"></i></a></p>
<p>Estimated time: <i aria-hidden="true" data-feather="clock"></i> 5 minutes</p>
<div class="text-end">
          <button role="button" aria-pressed="false" tabindex="0" id="expand-code" class="pull-right" data-expand="Expand All Solutions " data-collapse="Collapse All Solutions "> Expand All Solutions <i aria-hidden="true" data-feather="plus"></i></button>
        </div>
<div class="overview card">
<h2 class="card-header">Overview</h2>
<div class="row g-0">
<div class="col-md-4">
<div class="card-body">
<div class="inner">
<h3 class="card-title">Questions</h3>
<ul>
<li>What data are we using in this workshop?</li>
</ul>
</div>
</div>
</div>
<div class="col-md-8">
<div class="card-body">
<div class="inner bordered">
<h3 class="card-title">Objectives</h3>
<ul>
<li>Describe the biological pathway that will be referenced throughout
this tutorial</li>
</ul>
</div>
</div>
</div>
</div>
</div>
<section><h2 class="section-heading" id="introduction-to-the-data-used-in-this-workshop">Introduction to the data used in this workshop<a class="anchor" aria-label="anchor" href="#introduction-to-the-data-used-in-this-workshop"></a>
</h2>
<hr class="half-width">
<p>This tutorial uses the a well known and important signalling pathway
in the central nervous system (CNS) to illustrate some of the genome
browser tools and utility.</p>
<div class="section level3">
<h3 id="bdnf-and-trkb-signalling">BDNF and TrkB signalling<a class="anchor" aria-label="anchor" href="#bdnf-and-trkb-signalling"></a>
</h3>
<p>Brain Derived Neurotrophic factor (BDNF) protein is an important
neurotrophin responsible for regulating many aspects of growth and
development in different cells within the CNS. TrkB is an important
receptor that binds extracellular BDNF and propagates the intracellular
signalling response via a tyrosine kinase. This TrkB receptor protein is
encoded by the NTRK2 gene.</p>
<p>The NTRK2 gene expresses a number of different transcript variants in
different cell types. The most well studied of these is the full length
TrkB receptor referred to as TrkB, which is mainly expressed in neuronal
cell types. The other transcript variants all express the same exons
encoding the extracellular domain of the receptor (shown in the figure
here in green) but have truncated intracellular domains, which do not
include the tyrosine kinase domain and thus activate different
signalling pathways upon binding to BDNF. None of these truncated
protein products have been well studied, but the most highly expressed
receptor variant is known as TrkB-T1, and is known to be highly
expressed in astocytes.</p>
<figure><img src="../fig/introduction_TrkB-schema-eng.png" alt="Graphical representation of the BDNF and TrkB signalling pathway" class="figure mx-auto d-block"><div class="figcaption">Graphical representation of the BDNF and TrkB
signalling pathway</div>
</figure><p>Since the transcript variants are differently expressed in different
cell types within the CNS, the NTRK2 gene is a very useful example for
exploring cell type specific transcript expression in available public
data.</p>
</div>
<div class="section level3">
<h3 id="major-cns-cell-types">Major CNS cell types<a class="anchor" aria-label="anchor" href="#major-cns-cell-types"></a>
</h3>
<ul>
<li>Neuron (yellow cell in the image)</li>
<li>Astrocyte</li>
<li>Oligodendrocyte</li>
<li>Microglia</li>
<li>Ependymal</li>
</ul>
<figure><img src="../fig/introduction_CNScelltypes.jpg" alt="Image portraying the structure of different CNS cell types: neurons (yellow), astrocytes, microglia, oligodendrocytes and ependymal cells" class="figure mx-auto d-block"><div class="figcaption">Image portraying the structure of different CNS
cell types: neurons (yellow), astrocytes, microglia, oligodendrocytes
and ependymal cells</div>
</figure><div id="keypoints1" class="callout keypoints">
<div class="callout-square">
<i class="callout-icon" data-feather="key"></i>
</div>
<div class="callout-inner">
<h3 class="callout-title">Key Points</h3>
<div class="callout-content">
<ul>
<li>The NTRK2 gene is a useful example for exploring cell type specific
transcript expression in public data</li>
</ul>
</div>
</div>
</div>
<!--
Place links that you need to refer to multiple times across pages here. Delete
any links that you are not going to use.
 -->
</div>
</section></section><section id="aio-03_UCSC_setup"><p>Content from <a href="03_UCSC_setup.html">UCSC Genome Browser: Setup</a></p>
<hr>
<p>Last updated on 2025-07-25 |

        <a href="https://github.com/julia-broadbent/intro-to-genome-browsers/edit/main/episodes/03_UCSC_setup.md" class="external-link">Edit this page <i aria-hidden="true" data-feather="edit"></i></a></p>
<p>Estimated time: <i aria-hidden="true" data-feather="clock"></i> 26 minutes</p>
<div class="text-end">
          <button role="button" aria-pressed="false" tabindex="0" id="expand-code" class="pull-right" data-expand="Expand All Solutions " data-collapse="Collapse All Solutions "> Expand All Solutions <i aria-hidden="true" data-feather="plus"></i></button>
        </div>
<div class="overview card">
<h2 class="card-header">Overview</h2>
<div class="row g-0">
<div class="col-md-4">
<div class="card-body">
<div class="inner">
<h3 class="card-title">Questions</h3>
<ul>
<li>How do we navigate the web interface of the UCSC genome
browser?</li>
</ul>
</div>
</div>
</div>
<div class="col-md-8">
<div class="card-body">
<div class="inner bordered">
<h3 class="card-title">Objectives</h3>
<ul>
<li>Become familiar with the basic features of the UCSC genome
browser</li>
</ul>
</div>
</div>
</div>
</div>
</div>
<section><h2 class="section-heading" id="navigating-the-ucsc-browser">Navigating the UCSC browser<a class="anchor" aria-label="anchor" href="#navigating-the-ucsc-browser"></a>
</h2>
<hr class="half-width">
<p>Many of the tools that we will explore can be selected via multiple
different routes within the browser interface.</p>
<ul>
<li>Many tools can be accessed via the top toolbar on a pull down
list</li>
<li>Other tools can be accessed from within the browser window.</li>
</ul>
<p>In the following instructions, text written <code>like this</code> is
used to indicate successive lower levels from the pull down menu when
starting with the top toolbar. For example, the notation below indicates
that you should select <em>Genome Browser</em> from the top tool bar and
then click on <em>Reset all user settings</em>.</p>
<p><code>Toolbar   &gt;   Genome Browser  &gt;   Reset all user settings</code></p>
<div id="accordionSpoiler1" class="accordion spoiler-accordion accordion-flush">
<div class="accordion-item">
<button class="accordion-button spoiler-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapseSpoiler1" aria-expanded="false" aria-controls="collapseSpoiler1">
  <h3 class="accordion-header" id="headingSpoiler1">
  <div class="note-square"><i aria-hidden="true" class="callout-icon" data-feather="eye"></i></div>
  FURTHER HELP AND TRAINING
  </h3>
</button>
<div id="collapseSpoiler1" class="accordion-collapse collapse" data-bs-parent="#accordionSpoiler1" aria-labelledby="headingSpoiler1">
<div class="accordion-body">
<p>The UCSC genome browser is supported by a rich training resource
which has new material added regularly to the <a href="https://www.youtube.com/channel/UCQnUJepyNOw0p8s2otX4RYQ/videos" class="external-link">YouTube
channel</a>.</p>
<p>To access training and develop your skills further, go to:</p>
<p><code>Toolbar  &gt;  Help  &gt;  Training</code></p>
</div>
</div>
</div>
</div>
<div id="warning" class="callout caution">
<div class="callout-square">
<i class="callout-icon" data-feather="alert-triangle"></i>
</div>
<div id="warning" class="callout-inner">
<h3 class="callout-title">WARNING</h3>
<div class="callout-content">
<p>Weekly maintenance of the UCSC browser occurs at 2-3pm Sundays
Pacific time (7-8am Mondays AEST / 8-9am Mondays AEDT). During this time
the browser may be down for a few minutes.</p>
<p>To ensure uninterrupted browser services for your research during
UCSC server maintenance and power outages, bookmark one of the mirror
sites that replicates the UCSC genome browser.</p>
<p>More information and contacts for the UCSC genome browser can be
found <a href="http://seabass.mpipz.mpg.de/contacts.html" class="external-link">here</a>.</p>
</div>
</div>
</div>
</section><section><h2 class="section-heading" id="setting-up-your-browser">Setting up your browser<a class="anchor" aria-label="anchor" href="#setting-up-your-browser"></a>
</h2>
<hr class="half-width">
<div class="section level4">
<h4 id="open-the-browser-interface">1. Open the Browser interface<a class="anchor" aria-label="anchor" href="#open-the-browser-interface"></a>
</h4>
<p>Navigate to the <a href="https://genome.ucsc.edu/" class="external-link">UCSC genome
Browser</a> and sign in if you have an account.</p>
<figure><img src="../fig/03UCSC_UCSCHome.png" class="figure mx-auto d-block"></figure>
</div>
<div class="section level4">
<h4 id="reset-the-browser">2. Reset the browser<a class="anchor" aria-label="anchor" href="#reset-the-browser"></a>
</h4>
<p>To ensure we all see the same screen, select:</p>
<p><code>Toolbar  &gt;  Genome Browser  &gt;  Reset all user settings</code></p>
</div>
<div class="section level4">
<h4 id="select-and-open-the-human-genome-hg38-at-the-default-position">3. Select and open the human Genome Hg38 at the default
position<a class="anchor" aria-label="anchor" href="#select-and-open-the-human-genome-hg38-at-the-default-position"></a>
</h4>
<div class="tabs">
<nav><div id="nav-tab-1" class="nav nav-tabs" role="tablist">
<button class="nav-link active" id="nav-tab-1-1" data-bs-toggle="tab" data-bs-target="#nav-tabpanel-1-1" type="button" role="tab" aria-controls="nav-tabpanel-1-1" aria-selected="true">
  <h3 class="tab-header" id="nav-tab-heading-1-1">
  Go directly to the genome
  </h3>
</button>
<button class="nav-link" id="nav-tab-1-2" data-bs-toggle="tab" data-bs-target="#nav-tabpanel-1-2" type="button" role="tab" aria-controls="nav-tabpanel-1-2" aria-selected="false">
  <h3 class="tab-header" id="nav-tab-heading-1-2">
  …or go to the Genome gateway page
  </h3>
</button>
</div>
</nav><div id="nav-tabContent-1" class="tab-content">
<div id="nav-tabpanel-1-1" class="tab-pane show active" aria-labelledby="nav-tab-1-1" role="tabpanel">
<p><code>Toolbar  &gt;  Genomes  &gt;  HumanGRCh38</code></p>
</div>
<div id="nav-tabpanel-1-2" class="tab-pane" aria-labelledby="nav-tab-1-2" role="tabpanel">
<p><code>Toolbar  &gt;  Genomes</code></p>
<p>Ensure that GRCh38 is selected in ‘human assembly’ and click on the
blue <code>GO</code> box</p>
</div>
</div>
</div>
<figure><img src="../fig/03UCSC_UCSC_Hg38_opening_2021.png" alt="You should see a view of the browser similar to the image below, opening at a position on the X chromosome of Human genome version GRCh38 showing the gene model for the ACE2 gene. Some of the default tracks may have been updated since this screenshot was made." class="figure mx-auto d-block"><div class="figcaption">You should see a view of the browser similar to
the image below, opening at a position on the X chromosome of Human
genome version GRCh38 showing the gene model for the ACE2 gene. Some of
the default tracks may have been updated since this screenshot was
made.</div>
</figure>
</div>
<div class="section level4">
<h4 id="familiarise-yourself-with-the-main-areas-of-the-interface">4. Familiarise yourself with the main areas of the interface<a class="anchor" aria-label="anchor" href="#familiarise-yourself-with-the-main-areas-of-the-interface"></a>
</h4>
<div id="can-you-locate" class="callout checklist">
<div class="callout-square">
<i class="callout-icon" data-feather="check-square"></i>
</div>
<div id="can-you-locate" class="callout-inner">
<h3 class="callout-title">Can you locate:</h3>
<div class="callout-content">
<ul>
<li>The main Toolbar</li>
<li>Blue bar track collections (data of similar types are collected
together under the same ‘Blue bar’ heading). Scroll down to see
additional data collections and which ones are turned on as
default.</li>
<li>Genome species and version number</li>
<li>Position box</li>
<li>Navigation tool buttons</li>
<li>Chromosome ideogram</li>
<li>Genome view window</li>
<li>Pre loaded tracks, track titles:
<ul>
<li>The grey bars on the left of the genome view can be used for
selecting and configuring the tracks.</li>
<li>You can change the order of the tracks by dragging these grey bars
up and down.</li>
</ul>
</li>
<li>Turn tracks on and off:
<ul>
<li>You can hide tracks by right clicking on the grey bar or by turning
them off in the Blue bar collections.</li>
<li>You have to click on a ‘refresh’ button for changes to be reflected
in the genome view window.</li>
</ul>
</li>
<li>View the <strong>configuration page</strong> for one of the tracks.
The configuration page gives you a lot of information about the data
track and its colouring. You can open the configuration page for a track
by:
<ul>
<li>clicking on the grey bar for the track or,</li>
<li>clicking on the track title in the Blue bar collection. More
information and options are usually available by selecting the
configuration page for a track via the track title in the Blue Bar
collections.</li>
</ul>
</li>
<li>Select ‘resize’ button under the genome view to fit the genome view
window to your screen</li>
</ul>
</div>
</div>
</div>
</div>
<div class="section level4">
<h4 id="customise-your-view">5. Customise your view<a class="anchor" aria-label="anchor" href="#customise-your-view"></a>
</h4>
<p>Use the ‘Configure’ tool to change the font size to 12</p>
<div class="tabs">
<nav><div id="nav-tab-2" class="nav nav-tabs" role="tablist">
<button class="nav-link active" id="nav-tab-2-1" data-bs-toggle="tab" data-bs-target="#nav-tabpanel-2-1" type="button" role="tab" aria-controls="nav-tabpanel-2-1" aria-selected="true">
  <h3 class="tab-header" id="nav-tab-heading-2-1">
  Method #1
  </h3>
</button>
<button class="nav-link" id="nav-tab-2-2" data-bs-toggle="tab" data-bs-target="#nav-tabpanel-2-2" type="button" role="tab" aria-controls="nav-tabpanel-2-2" aria-selected="false">
  <h3 class="tab-header" id="nav-tab-heading-2-2">
  Method #2
  </h3>
</button>
</div>
</nav><div id="nav-tabContent-2" class="tab-content">
<div id="nav-tabpanel-2-1" class="tab-pane show active" aria-labelledby="nav-tab-2-1" role="tabpanel">
<p><code>Toolbar  &gt;  View  &gt;  Configure browser  &gt;  text size 12  &gt; submit</code></p>
</div>
<div id="nav-tabpanel-2-2" class="tab-pane" aria-labelledby="nav-tab-2-2" role="tabpanel">
<p>Click the <code>configure</code> button below the genome view
window</p>
<p>Ensure that GRCh38 is selected in ‘human assembly’ and click on the
blue <code>GO</code> box</p>
</div>
</div>
</div>
</div>
<div class="section level4">
<h4 id="practice-navigating-around-the-genome-view">6. Practice navigating around the genome view<a class="anchor" aria-label="anchor" href="#practice-navigating-around-the-genome-view"></a>
</h4>
<div id="can-you-practise" class="callout checklist">
<div class="callout-square">
<i class="callout-icon" data-feather="check-square"></i>
</div>
<div id="can-you-practise" class="callout-inner">
<h3 class="callout-title">Can you practise:</h3>
<div class="callout-content">
<ul>
<li>Moving left and right, using both the navigation buttons and your
mouse</li>
<li>Zooming in and out, using navigation buttons</li>
<li>Zooming in to a region of interest, using
<code>Drag-and-select</code>:
<ul>
<li>Using your mouse select a region of interest by clicking the ruler
(position track) at the very top of the genome view window.</li>
<li>This is also how to access the <em>highlight tool</em> which you
will use in a later exercise to highlight a region of interest.
<ul>
<li>Click on the down arrow next to the highlight colour to select a
different colour.</li>
</ul>
</li>
</ul>
</li>
</ul>
<figure><img src="../fig/03UCSC_drag-and-select.png" class="figure mx-auto d-block"></figure>
</div>
</div>
</div>
<p>Next, we will explore some tools and public datasets available in the
UCSC genome browser.</p>
<div id="keypoints1" class="callout keypoints">
<div class="callout-square">
<i class="callout-icon" data-feather="key"></i>
</div>
<div class="callout-inner">
<h3 class="callout-title">Key Points</h3>
<div class="callout-content">
<ul>
<li>The UCSC genome browser can be easily configured according to your
visual preferences and data needs</li>
</ul>
</div>
</div>
</div>
<!--
Place links that you need to refer to multiple times across pages here. Delete
any links that you are not going to use.
 -->
</div>
</section></section><section id="aio-04_UCSC_GeneModels"><p>Content from <a href="04_UCSC_GeneModels.html">UCSC Genome Browser: Understanding gene models</a></p>
<hr>
<p>Last updated on 2025-07-30 |

        <a href="https://github.com/julia-broadbent/intro-to-genome-browsers/edit/main/episodes/04_UCSC_GeneModels.md" class="external-link">Edit this page <i aria-hidden="true" data-feather="edit"></i></a></p>
<p>Estimated time: <i aria-hidden="true" data-feather="clock"></i> 33 minutes</p>
<div class="text-end">
          <button role="button" aria-pressed="false" tabindex="0" id="expand-code" class="pull-right" data-expand="Expand All Solutions " data-collapse="Collapse All Solutions "> Expand All Solutions <i aria-hidden="true" data-feather="plus"></i></button>
        </div>
<div class="overview card">
<h2 class="card-header">Overview</h2>
<div class="row g-0">
<div class="col-md-4">
<div class="card-body">
<div class="inner">
<h3 class="card-title">Questions</h3>
<ul>
<li>How do we interpret gene models represented in the UCSC genome
browser?</li>
</ul>
</div>
</div>
</div>
<div class="col-md-8">
<div class="card-body">
<div class="inner bordered">
<h3 class="card-title">Objectives</h3>
<p>Use the UCSC gene browser to:</p>
<ul>
<li>Explore features of particular chromosomal regions</li>
<li>Investigate specific genes as well as collections of genes</li>
<li>Search for locations of sequences and markers</li>
<li>Retrieve annotation information for specific regions or
genome-wide</li>
</ul>
</div>
</div>
</div>
</div>
</div>
<section><h2 class="section-heading" id="gene-model-representation">Gene model representation<a class="anchor" aria-label="anchor" href="#gene-model-representation"></a>
</h2>
<hr class="half-width">
<figure><img src="../fig/04UCSCGeneModels_gene_model.png" alt="The typical structure of a gene as represented in the UCSC genome browser" class="figure mx-auto d-block"><div class="figcaption">The typical structure of a gene as represented
in the UCSC genome browser</div>
</figure></section><section><h2 class="section-heading" id="ntrk2">NTRK2<a class="anchor" aria-label="anchor" href="#ntrk2"></a>
</h2>
<hr class="half-width">
<p>First we are going to familiarise ourselves with the gene model
representation of the different transcripts of NTRK2.</p>
<div class="section level4">
<h4 id="navigate-to-the-ntrk2-gene-position-in-grch38-and-view-the-gene-models-">1. Navigate to the NTRK2 gene position in GRCh38 and view the gene
models.<a class="anchor" aria-label="anchor" href="#navigate-to-the-ntrk2-gene-position-in-grch38-and-view-the-gene-models-"></a>
</h4>
<p>You can navigate to a different region of the genome by typing in the
position box.</p>
<div class="tabs">
<nav><div id="nav-tab-1" class="nav nav-tabs" role="tablist">
<button class="nav-link active" id="nav-tab-1-1" data-bs-toggle="tab" data-bs-target="#nav-tabpanel-1-1" type="button" role="tab" aria-controls="nav-tabpanel-1-1" aria-selected="true">
  <h3 class="tab-header" id="nav-tab-heading-1-1">
  Genomic location
  </h3>
</button>
<button class="nav-link" id="nav-tab-1-2" data-bs-toggle="tab" data-bs-target="#nav-tabpanel-1-2" type="button" role="tab" aria-controls="nav-tabpanel-1-2" aria-selected="false">
  <h3 class="tab-header" id="nav-tab-heading-1-2">
  Gene name
  </h3>
</button>
</div>
</nav><div id="nav-tabContent-1" class="tab-content">
<div id="nav-tabpanel-1-1" class="tab-pane show active" role="tabpanel" aria-labelledby="nav-tab-1-1">
<p>If you know the specific location you are interested in, type in the
location (e.g. <strong>chr9:84,665,760-85,030,334</strong>)</p>
</div>
<div id="nav-tabpanel-1-2" class="tab-pane" role="tabpanel" aria-labelledby="nav-tab-1-2">
<p>If you have a gene of interest, you can type in the gene name
(e.g. <strong>NTRK2</strong>).</p>
<p>Note the autocompleted suggestions that appear when you start typing.
You can select from one of the suggestions or click <code>go</code> and
select from a wider range of options.</p>
</div>
</div>
</div>
</div>
<div class="section level4">
<h4 id="hide-all-tracks">2. Hide all tracks<a class="anchor" aria-label="anchor" href="#hide-all-tracks"></a>
</h4>
<p>Select the <code>Hide all</code> button below the genome view.</p>
</div>
<div class="section level4">
<h4 id="turn-on-only-the-gencode-v36-genemodels-in-full-viewing-mode">3. Turn on only the <strong>Gencode V36</strong> Genemodels in
‘full’ viewing mode<a class="anchor" aria-label="anchor" href="#turn-on-only-the-gencode-v36-genemodels-in-full-viewing-mode"></a>
</h4>
<p>Select from the blue bar group labelled **Genes and Gene
predictions*’.</p>
</div>
<div class="section level4">
<h4 id="turn-on-the-conservation-track-to-full">4. Turn on the <strong>Conservation</strong> track to ‘full’<a class="anchor" aria-label="anchor" href="#turn-on-the-conservation-track-to-full"></a>
</h4>
<p>Don’t forget to click <code>refresh</code></p>
</div>
<div class="section level4">
<h4 id="zoom-out-and-scroll-across-the-whole-gene">5. Zoom out and scroll across the whole gene<a class="anchor" aria-label="anchor" href="#zoom-out-and-scroll-across-the-whole-gene"></a>
</h4>
<p>When you have navigated to the NTRK2 gene, zoom out until you can
view all of the 5’ UTRs and 3’ UTRs for all transcript variants for this
gene. Then drag the view left and right to center (like in Google maps)
or <em>drag and select</em> the region to center the gene in the Genome
view.</p>
<figure><img src="../fig/04UCSCGeneModels_NTRK2_GENCODE_V36.png" alt="You should now see something like this" class="figure mx-auto d-block"><div class="figcaption">You should now see something like this</div>
</figure><div id="challenge-1" class="callout discussion">
<div class="callout-square">
<i class="callout-icon" data-feather="message-circle"></i>
</div>
<div id="challenge-1" class="callout-inner">
<h3 class="callout-title">CHALLENGE 1</h3>
<div class="callout-content">
<p>Which strand is the gene encoded on / transcribed from? (+ or -
strand)</p>
</div>
</div>
</div>
<div id="challenge-2" class="callout discussion">
<div class="callout-square">
<i class="callout-icon" data-feather="message-circle"></i>
</div>
<div id="challenge-2" class="callout-inner">
<h3 class="callout-title">CHALLENGE 2</h3>
<div class="callout-content">
<p>Identify the exons, introns and UTRs.</p>
<p>Do regions of conservation only occur were there are coding
regions?</p>
</div>
</div>
</div>
<div id="challenge-3" class="callout discussion">
<div class="callout-square">
<i class="callout-icon" data-feather="message-circle"></i>
</div>
<div id="challenge-3" class="callout-inner">
<h3 class="callout-title">CHALLENGE 3</h3>
<div class="callout-content">
<p>How many different transcripts variants are there for this gene?</p>
<p>How do they differ?</p>
</div>
</div>
</div>
<p>Select a coding region (full height boxes) towards the 3’UTR of the
gene.</p>
<div id="challenge-4" class="callout discussion">
<div class="callout-square">
<i class="callout-icon" data-feather="message-circle"></i>
</div>
<div id="challenge-4" class="callout-inner">
<h3 class="callout-title">CHALLENGE 4</h3>
<div class="callout-content">
<p>Zoom in to the region until you can see the letters of the amino acid
sequence.</p>
<p>Why are some amino acid boxes red or green?</p>
</div>
</div>
</div>
<div id="challenge-5" class="callout discussion">
<div class="callout-square">
<i class="callout-icon" data-feather="message-circle"></i>
</div>
<div id="challenge-5" class="callout-inner">
<h3 class="callout-title">CHALLENGE 5</h3>
<div class="callout-content">
<p>Zoom in again until you can see each amino acid number.</p>
<p>Why do different transcripts have different amino acid numbers?</p>
</div>
</div>
</div>
<div id="note" class="callout">
<div class="callout-square">
<i class="callout-icon" data-feather="bell"></i>
</div>
<div id="note" class="callout-inner">
<h3 class="callout-title">NOTE</h3>
<div class="callout-content">
<p>Note that one of the transcript names is in white text with a black
background: this is the transcript you selected from the autocompleted
list or the search results.</p>
</div>
</div>
</div>
</div>
<div class="section level4">
<h4 id="change-the-view-settings-for-the-track">6. Change the <strong>view settings</strong> for the track<a class="anchor" aria-label="anchor" href="#change-the-view-settings-for-the-track"></a>
</h4>
<p>Right click on the track grey bar in the left of the genome window to
access view settings.</p>
<p>Switch between <code>dense</code> , <code>squish</code> ,
<code>pack</code> , and <code>full</code> to see how it changes the
representation of the models.</p>
</div>
<div class="section level4">
<h4 id="reveal-the-ensembl-id-for-each-transcript">7. Reveal the Ensembl ID for each transcript<a class="anchor" aria-label="anchor" href="#reveal-the-ensembl-id-for-each-transcript"></a>
</h4>
<p>Go to the configuration page for the <strong>GENCODE V36</strong>
track and change check the box to also reveal the <strong>Ensembl
ID</strong> in the label.</p>
<p>The transcript names are now too long to fit on the screen. You can
use the configuration page (like you did to chane the font size at the
beginning of the workshop) and change the number of characters in the
label so that you can see the entire transcript label.</p>
</div>
<div class="section level3">
<h3 id="check-your-understanding">Check your understanding<a class="anchor" aria-label="anchor" href="#check-your-understanding"></a>
</h3>
<div id="question-1" class="callout challenge">
<div class="callout-square">
<i class="callout-icon" data-feather="zap"></i>
</div>
<div id="question-1" class="callout-inner">
<h3 class="callout-title">Question 1</h3>
<div class="callout-content">
<p>Which transcript encodes the shortest amino acid sequence?</p>
<figure><img src="../fig/04UCSCGeneModels_quizQ1.jpg" class="figure mx-auto d-block"></figure>
</div>
</div>
</div>
<div id="accordionSolution1" class="accordion challenge-accordion accordion-flush">
<div class="accordion-item">
<button class="accordion-button solution-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapseSolution1" aria-expanded="false" aria-controls="collapseSolution1">
  <h4 class="accordion-header" id="headingSolution1">
  Answer
  </h4>
</button>
<div id="collapseSolution1" class="accordion-collapse collapse" aria-labelledby="headingSolution1" data-bs-parent="#accordionSolution1">
<div class="accordion-body">
<p><strong>ENST00000352327.5</strong></p>
<p>This transcript does not include one of the large coding regions and
the coding region in the terminal exon is also slightly shorter.</p>
</div>
</div>
</div>
</div>
<div id="question-2" class="callout challenge">
<div class="callout-square">
<i class="callout-icon" data-feather="zap"></i>
</div>
<div id="question-2" class="callout-inner">
<h3 class="callout-title">Question 2</h3>
<div class="callout-content">
<p>Which transcript has the longest 3’UTR?</p>
<figure><img src="../fig/04UCSCGeneModels_quizQ2.jpg" class="figure mx-auto d-block"></figure>
</div>
</div>
</div>
<div id="accordionSolution2" class="accordion challenge-accordion accordion-flush">
<div class="accordion-item">
<button class="accordion-button solution-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapseSolution2" aria-expanded="false" aria-controls="collapseSolution2">
  <h4 class="accordion-header" id="headingSolution2">
  Answer
  </h4>
</button>
<div id="collapseSolution2" class="accordion-collapse collapse" aria-labelledby="headingSolution2" data-bs-parent="#accordionSolution2">
<div class="accordion-body">
<p><strong>ENST00000396976.6</strong></p>
<p>Although the last two exons of ENST00000396972.2 are UTR rather than
coding sequence, it is still not as long as the 3’UTR of
ENST00000396976.6</p>
</div>
</div>
</div>
</div>
<div id="question-3" class="callout challenge">
<div class="callout-square">
<i class="callout-icon" data-feather="zap"></i>
</div>
<div id="question-3" class="callout-inner">
<h3 class="callout-title">Question 3</h3>
<div class="callout-content">
<p>Which transcripts appear to encode the same protein product?</p>
<figure><img src="../fig/04UCSCGeneModels_quizQ3.jpg" class="figure mx-auto d-block"></figure>
</div>
</div>
</div>
<div id="accordionSolution3" class="accordion challenge-accordion accordion-flush">
<div class="accordion-item">
<button class="accordion-button solution-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapseSolution3" aria-expanded="false" aria-controls="collapseSolution3">
  <h4 class="accordion-header" id="headingSolution3">
  Answer
  </h4>
</button>
<div id="collapseSolution3" class="accordion-collapse collapse" aria-labelledby="headingSolution3" data-bs-parent="#accordionSolution3">
<div class="accordion-body">
<p><strong>ENST00000420986.6, ENST00000280352.13 and
ENST00000393047.8</strong></p>
<p>Notice the height of the boxes for the first three exons:
ENST00000532163.5 appears to encode a different CDS from the other
transcripts.</p>
</div>
</div>
</div>
</div>
<div id="question-4" class="callout challenge">
<div class="callout-square">
<i class="callout-icon" data-feather="zap"></i>
</div>
<div id="question-4" class="callout-inner">
<h3 class="callout-title">Question 4</h3>
<div class="callout-content">
<p>Which transcript has the longest 5’UTR?</p>
<figure><img src="../fig/04UCSCGeneModels_quizQ4.jpg" class="figure mx-auto d-block"></figure>
</div>
</div>
</div>
<div id="accordionSolution4" class="accordion challenge-accordion accordion-flush">
<div class="accordion-item">
<button class="accordion-button solution-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapseSolution4" aria-expanded="false" aria-controls="collapseSolution4">
  <h4 class="accordion-header" id="headingSolution4">
  Answer
  </h4>
</button>
<div id="collapseSolution4" class="accordion-collapse collapse" aria-labelledby="headingSolution4" data-bs-parent="#accordionSolution4">
<div class="accordion-body">
<p><strong>ENST00000532681.5</strong></p>
<p>This transcript is encoded on the reverse strand.</p>
</div>
</div>
</div>
</div>
<div id="question-5" class="callout challenge">
<div class="callout-square">
<i class="callout-icon" data-feather="zap"></i>
</div>
<div id="question-5" class="callout-inner">
<h3 class="callout-title">Question 5</h3>
<div class="callout-content">
<p>Which transcript has the longest CDS?</p>
<figure><img src="../fig/04UCSCGeneModels_quizQ5.jpg" class="figure mx-auto d-block"></figure>
</div>
</div>
</div>
<div id="accordionSolution5" class="accordion challenge-accordion accordion-flush">
<div class="accordion-item">
<button class="accordion-button solution-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapseSolution5" aria-expanded="false" aria-controls="collapseSolution5">
  <h4 class="accordion-header" id="headingSolution5">
  Answer
  </h4>
</button>
<div id="collapseSolution5" class="accordion-collapse collapse" aria-labelledby="headingSolution5" data-bs-parent="#accordionSolution5">
<div class="accordion-body">
<p><strong>ENST00000253801.7</strong></p>
<p>This transcript is encoded on the forward strand.</p>
</div>
</div>
</div>
</div>
<div id="question-6" class="callout challenge">
<div class="callout-square">
<i class="callout-icon" data-feather="zap"></i>
</div>
<div id="question-6" class="callout-inner">
<h3 class="callout-title">Question 6</h3>
<div class="callout-content">
<p>Which transcript has the longest 5’UTR?</p>
<figure><img src="../fig/04UCSCGeneModels_quizQ6.jpg" class="figure mx-auto d-block"></figure>
</div>
</div>
</div>
<div id="accordionSolution6" class="accordion challenge-accordion accordion-flush">
<div class="accordion-item">
<button class="accordion-button solution-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapseSolution6" aria-expanded="false" aria-controls="collapseSolution6">
  <h4 class="accordion-header" id="headingSolution6">
  Answer
  </h4>
</button>
<div id="collapseSolution6" class="accordion-collapse collapse" aria-labelledby="headingSolution6" data-bs-parent="#accordionSolution6">
<div class="accordion-body">
<p><strong>ENST00000589334.5</strong></p>
<p>This transcript is encoded on the reverse strand.</p>
</div>
</div>
</div>
</div>
</div>
</section><section><h2 class="section-heading" id="bdnf">BDNF<a class="anchor" aria-label="anchor" href="#bdnf"></a>
</h2>
<hr class="half-width">
<p>Now we will look at the gene model for BDNF in the same genome. There
are some differences that enable us to demonstrate some more tools.</p>
<div class="section level4">
<h4 id="navigate-to-the-bdnf-gene-position-in-grch38">1. Navigate to the BDNF gene position in GRCh38<a class="anchor" aria-label="anchor" href="#navigate-to-the-bdnf-gene-position-in-grch38"></a>
</h4>
<figure><img src="../fig/04UCSCgenemodels_BDNF_genemodel_2021.png" class="figure mx-auto d-block"></figure><ul>
<li>Note that there are blue transcript models encoded on the - strand
and green BDNS-AS transcript models on the + strand. BDNF-AS is the
antisense gene.</li>
<li>Colouring information is specific for each track and can be obtained
from the configuration page. Below is the colouring legend for the
GENCODE V36 track.</li>
</ul>
<figure><img src="../fig/04UCSCgenemodels_Gencode_V36_colour.png" class="figure mx-auto d-block"></figure>
</div>
<div class="section level4">
<h4 id="flip-the-orientation-of-the-gene">2. Flip the orientation of the gene<a class="anchor" aria-label="anchor" href="#flip-the-orientation-of-the-gene"></a>
</h4>
<p>Since the convention is to display genes in the 5’ to 3’ orientation,
it can be useful for our own interpretation, and also for presentation
purposes, to flip the orientation of a gene when viewing it in a Genome
Browser.</p>
<p>To flip the orientation of the gene, use the <code>reverse</code>
button under the genome view window.</p>
</div>
<div class="section level4">
<h4 id="apply-the-multi-region-view-from-the-main-tool-bar-view-options">3. Apply the Multi-Region view from the main tool bar view
options<a class="anchor" aria-label="anchor" href="#apply-the-multi-region-view-from-the-main-tool-bar-view-options"></a>
</h4>
<p>When a gene has many large introns taking up a lot of white space in
an image, it can be difficult to see if exons in different transcript
models or other data tracks align. The <strong>Multi-Region</strong>
view tool can be used to fold the intronic regions out of the view like
a concertina. The Broswer selects which region to fold out based on the
gene model track(s) that you have turned on at the time.</p>
<p><code>Toolbar  &gt;  View  &gt;  Multi-Region  &gt;</code> select
<code>Show exons using GENCODE V36</code></p>
<figure><img src="../fig/04UCSCgenemodels_BDNF_multiregion_2021.png" alt="It is now a lot easier to view a number of interesting features in the BDNF transcript models" class="figure mx-auto d-block"><div class="figcaption">It is now a lot easier to view a number of
interesting features in the BDNF transcript models</div>
</figure><p>Notice:</p>
<ul>
<li>The transcript variants for the BDNF vary mostly in the genomic
position of the 5’UTR.</li>
<li>The noncoding AS-BDNF gene transcript includes a region that would
be antisense to the coding BDNF transcript.</li>
</ul>
<p>You may find that using the multi-region tool facilitates
visualisation and interpretation of gene expression data later in the
workshop.</p>
<div id="keypoints1" class="callout keypoints">
<div class="callout-square">
<i class="callout-icon" data-feather="key"></i>
</div>
<div class="callout-inner">
<h3 class="callout-title">Key Points</h3>
<div class="callout-content">
<ul>
<li>The UCSC genome browser graphically represents key elements of gene
transcripts, including exons, introns, and untranslated regions</li>
<li>Different settings and tools can be used to configure the browser to
more easily investigate specific features of a gene</li>
</ul>
</div>
</div>
</div>
<!--
Place links that you need to refer to multiple times across pages here. Delete
any links that you are not going to use.
 -->
</div>
</section></section><section id="aio-05_BLAT"><p>Content from <a href="05_BLAT.html">UCSC Genome Browser: BLAT tool</a></p>
<hr>
<p>Last updated on 2025-07-30 |

        <a href="https://github.com/julia-broadbent/intro-to-genome-browsers/edit/main/episodes/05_BLAT.md" class="external-link">Edit this page <i aria-hidden="true" data-feather="edit"></i></a></p>
<p>Estimated time: <i aria-hidden="true" data-feather="clock"></i> 21 minutes</p>
<div class="text-end">
          <button role="button" aria-pressed="false" tabindex="0" id="expand-code" class="pull-right" data-expand="Expand All Solutions " data-collapse="Collapse All Solutions "> Expand All Solutions <i aria-hidden="true" data-feather="plus"></i></button>
        </div>
<div class="overview card">
<h2 class="card-header">Overview</h2>
<div class="row g-0">
<div class="col-md-4">
<div class="card-body">
<div class="inner">
<h3 class="card-title">Questions</h3>
<ul>
<li>What is the BLAT tool and how do we use it?</li>
</ul>
</div>
</div>
</div>
<div class="col-md-8">
<div class="card-body">
<div class="inner bordered">
<h3 class="card-title">Objectives</h3>
<ul>
<li>Use the UCSC BLAT tool to compare the similarity of a short probe
sequence to the human genome and identify the genomic transcript each
probe is most likely to detect.</li>
</ul>
</div>
</div>
</div>
</div>
</div>
<section><h2 class="section-heading" id="the-blat-tool">The BLAT tool<a class="anchor" aria-label="anchor" href="#the-blat-tool"></a>
</h2>
<hr class="half-width">
<p>The <a href="https://genome.ucsc.edu/FAQ/FAQblat.html" class="external-link">BLAT tool</a>
is a sequence similarity tool similar to BLAST. It can quickly identify
region(s) of homology between a genome and a sequence of interest. Due
to the presence of orthologs and paralogs, a target sequence may have
similarity to more than one region in the genome.</p>
</section><section><h2 class="section-heading" id="a-note-on-microarray-expression-data">A note on microarray expression data<a class="anchor" aria-label="anchor" href="#a-note-on-microarray-expression-data"></a>
</h2>
<hr class="half-width">
<p>Microarray expression data is not commonly used now, but some of the
data generated from large, well-orchestrated studies still provide
valuable information to researchers. Microarray probes, like <em>in
situ</em> hybridisation probes, target a small region of the RNA and do
not measure the whole RNA transcript. If you are measuring gene
expression, it is important to know exactly which region of the gene you
are detecting.</p>
</section><section><h2 class="section-heading" id="exercise">Exercise<a class="anchor" aria-label="anchor" href="#exercise"></a>
</h2>
<hr class="half-width">
<p>In this exercise, we will employ the BLAT tool to map the sequences
of two different expression probes to their target regions and determine
which NTRK2 gene transcripts the probes are likely to detect.</p>
<p>The study was the <a href="http://human.brain-map.org/" class="external-link">Human Brain
gene expression atlas</a> generated by the <a href="https://alleninstitute.org/about/" class="external-link">Allen Institute</a>. Below are
sequences of two hybridisation probes that were use in a microarray used
to detect expression of the gene NTRK2. These two probes result in very
different hybridisation and expression patterns across different regions
of the brain. As we observed in the previous exercise, NTRK2 has a
number of different transcript variants.</p>
<blockquote>
<p><strong>Our question: are these probes are detecting different or
multiple transcripts of NTRK2, and if yes, which ones?</strong></p>
</blockquote>
<div class="section level3">
<h3 id="the-probes">The probes<a class="anchor" aria-label="anchor" href="#the-probes"></a>
</h3>
<p>The images below are of one of the six donors included in the atlas,
and typical of the expression pattern for NTRK2. These images are taken
from the <a href="http://human.brain-map.org/microarray/gene/show/4884" class="external-link">NTRK2 gene
page of Human Brain Atlas</a>.</p>
<p>Most obvious in the images is the high level of expression signal
using Probe A_23_P216779 and low level for A_24_P343559 in the corpus
callosum (CC), which is a region of white matter in the brain with
relatively few neurons and relatively high proportion of myelinating
oligodendrocytes. This expression profile is reversed in the the
cortical regions, eg. frontal lobe (FL) and parietal lobe (PL), which
have a relatively high density of neuronal cells.</p>
<div class="section level4">
<h4 id="ntrk2-probe-a_23_p216779">NTRK2 Probe A_23_P216779<a class="anchor" aria-label="anchor" href="#ntrk2-probe-a_23_p216779"></a>
</h4>
<p><strong>Sequence:</strong>
TTCTATACTCTAATCAGCACTGAATTCAGAGGGTTTGACTTTTTCATCTATAACACAGTG</p>
<figure><img src="../fig/05BLAT_ProbeA23_ABA_P216779.png" alt="Z score of expression level in Human brain (blue = low expression, red = high expression)" class="figure mx-auto d-block"><div class="figcaption">Z score of expression level in Human brain (blue
= low expression, red = high expression)</div>
</figure>
</div>
<div class="section level4">
<h4 id="ntrk2-probe-a_24_p343559">NTRK2 Probe A_24_P343559<a class="anchor" aria-label="anchor" href="#ntrk2-probe-a_24_p343559"></a>
</h4>
<p><strong>Sequence:</strong>
AAGCTGCTCTCCTTCACTCTGACAGTATTAACATCAAAGACTCCGAGAAGCTCTCGAGGG</p>
<figure><img src="../fig/05BLAT_ProbeA24_ABA_P343559.png" alt="Z score of expression level in Human brain (blue = low expression, red = high expression)" class="figure mx-auto d-block"><div class="figcaption">Z score of expression level in Human brain (blue
= low expression, red = high expression)</div>
</figure>
</div>
</div>
<div class="section level3">
<h3 id="your-task">Your task<a class="anchor" aria-label="anchor" href="#your-task"></a>
</h3>
<div class="section level4">
<h4 id="use-the-blat-tool-to-find-region-of-homology">1. Use the BLAT tool to find region of homology<a class="anchor" aria-label="anchor" href="#use-the-blat-tool-to-find-region-of-homology"></a>
</h4>
<ol style="list-style-type: lower-alpha">
<li><p>Select <code>Toolbar  &gt;  tools   &gt;  blat</code></p></li>
<li><p>Copy the sequence of the first probe above and paste into the
search box</p></li>
<li><p>Select the human GRCh38 for ‘Assembly’ and click
<code>Submit</code></p></li>
</ol>
<p>Probe A_23_P216779 returns 2 hits for different chromosomes. One of
these has 100% homology over the whole 60 base sequence, the other has
87% homology over a 24 base region.</p>
<ol start="4" style="list-style-type: lower-alpha">
<li><p>Copy and paste the probe name to use as the label for the
<strong>Custom track name</strong> and <strong>Custom track
description</strong> and click on
<code>Build a custom track with these results</code> . It is not
necessary to build a custom track - you could simply click on the
<code>browser</code> link to view the results, but creating a ‘custom
track’ from the BLAT result enables you to give it a unique name. This
is important if you are doing multiple BLAT searches as you wont be able
to tell which one is which.</p></li>
<li><p>Select <code>browser</code> option for the hit with the highest
homology to view the result. Observe which region of the NTRK2 gene this
probe will target. Zoom out for context.</p></li>
<li><p>Note that a new <strong>Blue bar</strong> heading has appeared
for your custom tracks.</p></li>
<li><p>Repeat for the other probe sequence.</p></li>
</ol>
</div>
<div class="section level4">
<h4 id="use-the-highlight-tool-to-keep-track-of-region-of-interest-in-the-genome-view-">2. Use the ‘highlight’ tool to keep track of region of interest in
the Genome view.<a class="anchor" aria-label="anchor" href="#use-the-highlight-tool-to-keep-track-of-region-of-interest-in-the-genome-view-"></a>
</h4>
<ol style="list-style-type: lower-alpha">
<li><p>It is easy to lose track of a region you are investigating when
navigating around the genome in a browser. We are going to highlight
each region of probe homology within the NTRK2 gene, using a different
colour for each probe. Highlight is also useful if you have lots of
different tracks loaded and you want to check that a feature on one
track lines up with another.</p></li>
<li><p>Using your mouse, select in the position track at the top to
activate <strong>Drag-and-select</strong>. Select only the region of
homology for each probe within the NTRK2 gene and use a different
highlight for each region.</p></li>
<li><p>Zoom out to view the whole gene again.</p></li>
</ol>
<div id="discussion1" class="callout discussion">
<div class="callout-square">
<i class="callout-icon" data-feather="message-circle"></i>
</div>
<div class="callout-inner">
<h3 class="callout-title">Challenge</h3>
<div class="callout-content">
<p>Do the probes detect coding regions of the NTRK2 gene?</p>
</div>
</div>
</div>
<div id="discussion2" class="callout discussion">
<div class="callout-square">
<i class="callout-icon" data-feather="message-circle"></i>
</div>
<div class="callout-inner">
<h3 class="callout-title">Challenge</h3>
<div class="callout-content">
<p>Are the probes likely to detect different transcripts?</p>
</div>
</div>
</div>
</div>
<div class="section level4">
<h4 id="use-multiregion-view-to-make-it-easier-to-compare-coding-regions-of-different-transcripts">3. Use ‘Multiregion view’ to make it easier to compare coding
regions of different transcripts<a class="anchor" aria-label="anchor" href="#use-multiregion-view-to-make-it-easier-to-compare-coding-regions-of-different-transcripts"></a>
</h4>
<p><code>Toolbar  &gt;  View  &gt;  Multi-Region</code> select
<code>Show exons using GENCODE V36</code></p>
<blockquote>
<p>We have created a ‘public session’ of the BLAT NTRK2 exercise. You
can view this from the link in the sessions.</p>
<p><code>Toolbar  &gt;  My data   &gt;  Public session  &gt;</code>
search for <strong>hg38_NTRK2_blat_probes</strong></p>
</blockquote>
<div id="keypoints1" class="callout keypoints">
<div class="callout-square">
<i class="callout-icon" data-feather="key"></i>
</div>
<div class="callout-inner">
<h3 class="callout-title">Key Points</h3>
<div class="callout-content">
<ul>
<li>We can use the UCSC <a href="https://genome.ucsc.edu/FAQ/FAQblat.html" class="external-link">BLAT tool</a> to
identify region(s) of sequence similarity between a genome and a
sequence of interest.</li>
</ul>
</div>
</div>
</div>
<!--
Place links that you need to refer to multiple times across pages here. Delete
any links that you are not going to use.
 -->
</div>
</div>
</section></section><section id="aio-06_gene_expression_data"><p>Content from <a href="06_gene_expression_data.html">UCSC Genome Browser: Gene expression data</a></p>
<hr>
<p>Last updated on 2025-07-30 |

        <a href="https://github.com/julia-broadbent/intro-to-genome-browsers/edit/main/episodes/06_gene_expression_data.md" class="external-link">Edit this page <i aria-hidden="true" data-feather="edit"></i></a></p>
<p>Estimated time: <i aria-hidden="true" data-feather="clock"></i> 33 minutes</p>
<div class="text-end">
          <button role="button" aria-pressed="false" tabindex="0" id="expand-code" class="pull-right" data-expand="Expand All Solutions " data-collapse="Collapse All Solutions "> Expand All Solutions <i aria-hidden="true" data-feather="plus"></i></button>
        </div>
<div class="overview card">
<h2 class="card-header">Overview</h2>
<div class="row g-0">
<div class="col-md-4">
<div class="card-body">
<div class="inner">
<h3 class="card-title">Questions</h3>
<ul>
<li>How can we explore gene expression data in the UCSC genome
browser?</li>
</ul>
</div>
</div>
</div>
<div class="col-md-8">
<div class="card-body">
<div class="inner bordered">
<h3 class="card-title">Objectives</h3>
<p>Use the UCSC gene browser to:</p>
<ul>
<li>Interpret gene expression data from multiple sources</li>
<li>Compare a region of one genome to genomes of other species</li>
</ul>
</div>
</div>
</div>
</div>
</div>
<figure><img src="../fig/06geneexpression_coverage_plot.png" class="figure mx-auto d-block"></figure><section><h2 class="section-heading" id="gtex-data">GTEX data<a class="anchor" aria-label="anchor" href="#gtex-data"></a>
</h2>
<hr class="half-width">
<p>Human tissue specific expression data from the <a href="https://gtexportal.org/home/" class="external-link">GTEX project</a> is available in the
UCSC genome browser.</p>
<ul>
<li>Gene level expression data from GTEx V6 (570) donors, and GTEx V8
(948) donors can be turned on from Blue bar title for more detailed
configuration page. These are displayed as coloured bar plots.</li>
<li>Transcript level data is also available for GTEx V6, this is also
displayed as bar plots.</li>
<li>Transcript level expression data for GTEx V6 is available as
coverage plots and is accessed from the <strong>Track hubs</strong>.
<ul>
<li>Under the genome view window, click on the <code>track hubs</code>,
or access from the <code>Toolbar</code> under <code>My data</code>
</li>
<li>Scroll down and select <code>GTEx RNA-seq Signal Hub</code>. This
can be viewed in either hg19 or hg38 so can be compared with a wide
variety of other datasets.</li>
</ul>
<img src="../fig/06geneexpression_trackhubs_GTEx.png" class="figure">
</li>
</ul>
<p>Default settings:</p>
<ul>
<li><p>All the available data from one individual only is loaded. Data
from other subjects in the study can be loaded as desired. For example,
you could load all available samples for one tissue region
only.</p></li>
<li><p>The data is <em>autoscale to data view</em> with a track height
or 12 pixels for each samples. You can change the height of the track or
add a data transformation.</p></li>
<li>
<p>The default points plot can be converted into a density plot by
clicking on the track title.</p>
<figure><img src="../fig/06geneexpression_GTEx_V6_settings.png" class="figure mx-auto d-block"></figure>
</li>
</ul>
<div class="section level3">
<h3 id="your-task">Your task<a class="anchor" aria-label="anchor" href="#your-task"></a>
</h3>
<p>Using the selection matrix for female donors aged 2-49 years,
deselect the default samples and select only Brain cortex and Pancreas
samples.</p>
<p>Navigate to the location for the gene MYRF. You can increase the
height of the datatrack to improve visualisation.</p>
<div id="challenge-1" class="callout discussion">
<div class="callout-square">
<i class="callout-icon" data-feather="message-circle"></i>
</div>
<div id="challenge-1" class="callout-inner">
<h3 class="callout-title">CHALLENGE 1</h3>
<div class="callout-content">
<p>Can you locate an exon in the MYRF gene that is present in
transcripts expressed in the brain but not in the pancreas?</p>
</div>
</div>
</div>
<div id="challenge-2" class="callout discussion">
<div class="callout-square">
<i class="callout-icon" data-feather="message-circle"></i>
</div>
<div id="challenge-2" class="callout-inner">
<h3 class="callout-title">CHALLENGE 2</h3>
<div class="callout-content">
<p>Does this alternative splicing event result in a frame shift of the
coding sequence?</p>
</div>
</div>
</div>
<div id="challenge-3" class="callout discussion">
<div class="callout-square">
<i class="callout-icon" data-feather="message-circle"></i>
</div>
<div id="challenge-3" class="callout-inner">
<h3 class="callout-title">CHALLENGE 3</h3>
<div class="callout-content">
<p>How many amino acids are there in the protein products for each MYRF
transcript?</p>
</div>
</div>
</div>
</div>
</section><section><h2 class="section-heading" id="facs-data">FACS data<a class="anchor" aria-label="anchor" href="#facs-data"></a>
</h2>
<hr class="half-width">
<p>The FACS derived data from the <a href="https://tabula-muris.ds.czbiohub.org/" class="external-link">Tabular Muris</a> cell type
data can be visualised as a coverage plot</p>
<div class="section level3">
<h3 id="your-task-1">Your task<a class="anchor" aria-label="anchor" href="#your-task-1"></a>
</h3>
<ol style="list-style-type: decimal">
<li>Start at the view of the NTRK2 gene in the human genome and navigate
to the Ntrk2 gene in the mouse genome using the <strong>View in other
genomes tool</strong>.</li>
</ol>
<p><code>Toolbar  &gt;  View  &gt;  In Other Genomes</code>
(Convert)</p>
<ol start="2" style="list-style-type: decimal">
<li><p>Select <code>New Genome:Mouse</code>,
<code>New Assembly:GRC38/mm10</code>, click on
<code>Submit</code></p></li>
<li><p>Select the region with the greatest homology</p></li>
<li><p>Click on the bar chart icon for the Tabular muris data in the
default view to see a summary bar chart of the cell type data.</p></li>
<li>
<p>To see a coverage plot of the expression data, we have to
configure the Tabular Muris track by selecting it from the blue bar
collection.</p>
<ol style="list-style-type: lower-alpha">
<li>Hide <code>Cell expression</code>
</li>
<li>Select <code>Genome coverage</code> to full</li>
<li>Select <code>submit</code>
</li>
</ol>
</li>
</ol>
<p><em>This can look a bit overwhelming as there are many tracks and the
default track height is set very high. </em> <em>But it is easy to
simplify it by focusing on a few cell types of interest.</em></p>
<ol start="6" style="list-style-type: decimal">
<li>Right click on the grey bar to <code>configure the track set</code>
<ol style="list-style-type: lower-alpha">
<li>Change <code>Track height</code> to 30</li>
<li>for <code>Data view scaling</code> select
<code>group auto-scale</code>
</li>
<li>clear all the subtracks and then manually select only these few cell
types of interest:</li>
</ol>
<ul>
<li>astrocyte Cv</li>
<li>Bergmann glial Cv</li>
<li>microglia Cv</li>
<li>neuron Cv</li>
<li>oligodendrocyte Cv</li>
<li>OPC Cv</li>
</ul>
</li>
</ol>
<div id="challenge-4" class="callout discussion">
<div class="callout-square">
<i class="callout-icon" data-feather="message-circle"></i>
</div>
<div id="challenge-4" class="callout-inner">
<h3 class="callout-title">CHALLENGE 4</h3>
<div class="callout-content">
<p>Which cell type has the highest level expression of Ntrk2 in this
dataset?</p>
</div>
</div>
</div>
<ol start="7" style="list-style-type: decimal">
<li><p>Change the <code>Data view scaling</code> from autoscale to
dataview.</p></li>
<li><p>Export a PDF image of the genome view:
<code>Toolbar  &gt;  View  &gt;  PDF/PS</code> select
<code>Download the current browser graphic in PDF</code></p></li>
</ol>
<figure><img src="../fig/06geneexpression_mm10_Ntrk2.png" class="figure mx-auto d-block"></figure><div id="challenge-5" class="callout discussion">
<div class="callout-square">
<i class="callout-icon" data-feather="message-circle"></i>
</div>
<div id="challenge-5" class="callout-inner">
<h3 class="callout-title">CHALLENGE 5</h3>
<div class="callout-content">
<p>Which cell type(s) express the long and short transcripts of
Ntrk2?</p>
</div>
</div>
</div>
</div>
</section><section><h2 class="section-heading" id="linnarsson-lab-data">Linnarsson lab data<a class="anchor" aria-label="anchor" href="#linnarsson-lab-data"></a>
</h2>
<hr class="half-width">
<p>Mouse CNS cell type expression data can also be validated using an
independent single cell dataset of mouse cortex from the <a href="http://linnarssonlab.org/" class="external-link">Linnarsson lab</a>.</p>
<p>The data that is publicly available for viewing in the UCCS genome
browser is not housed in the UCSC genome browser. You must first access
it from the the Linnarsson lab data page.</p>
<p>This RNAseq data is stranded, meaning you can see if the transcript
data is from the + or - strand.</p>
<div class="section level3">
<h3 id="your-task-2">Your task<a class="anchor" aria-label="anchor" href="#your-task-2"></a>
</h3>
<ol style="list-style-type: decimal">
<li><p>Click <a href="http://linnarssonlab.org/cortex/" class="external-link">here</a> for the
Linnarsson lab public data page for this dataset where you can search
for cell expression profiles for individual genes.</p></li>
<li><p>Click on the <code>Browse the genome</code> blue text near the
bottom of the page.</p></li>
</ol>
<p>This loads 18 different tracks, one for each cell type investigated.
The default setting for expression range is quite high and most gene
expression is not observed with these settings. Each track must be
configured individually rather than as a group, which takes a lot of
time.</p>
<p>We have created a version of this data as a public session in the
UCSC genome browser, where each track is autoscaled which can make it
quicker to determine which expression range would be ideal for
visualising the expression of an individual gene. The data is also
viewed using ‘Multi-Region’ which hides the introns in the gene models.
The session is illustrated in the screen shot below and you can access
this custom track set by clicking on the title in blue text. It is also
a publicly shared session called <strong>mm10 Linnarsson Celltype
autoscale</strong> which you can access from the database of public
sessions.</p>
<figure><img src="../fig/06geneexpression_linnarsson_ntrk2.png" alt="Linnarsson lab mouse cortex single cell data as autoscaled datatracks" class="figure mx-auto d-block"><div class="figcaption"><a href="https://genome.ucsc.edu/cgi-bin/hgTracks?db=mm10&amp;lastVirtModeType=exonMostly&amp;lastVirtModeExtraState=&amp;emGeneTable=knownGene&amp;virtModeType=exonMostly&amp;virtMode=1&amp;nonVirtPosition=chr13%3A58807697%2D59133970&amp;position=chr13%3A58807697%2D59133970&amp;hgsid=1130613507_A81QmYr4dhsQBuI4PbXkGTxoEaZH" class="external-link">Linnarsson
lab mouse cortex single cell data as autoscaled datatracks</a></div>
</figure><div id="challenge-6" class="callout discussion">
<div class="callout-square">
<i class="callout-icon" data-feather="message-circle"></i>
</div>
<div id="challenge-6" class="callout-inner">
<h3 class="callout-title">CHALLENGE 6</h3>
<div class="callout-content">
<p>Select two or three cell types and adjust the scale to best reflect
differences in gene expression of Ntrk2 between these cells.</p>
<p>Save this session and share it.</p>
</div>
</div>
</div>
<div id="keypoints1" class="callout keypoints">
<div class="callout-square">
<i class="callout-icon" data-feather="key"></i>
</div>
<div class="callout-inner">
<h3 class="callout-title">Key Points</h3>
<div class="callout-content">
<ul>
<li>The UCSC genome browser allows you to compare gene expression data
from multiple sources and different species</li>
</ul>
</div>
</div>
</div>
<!--
Place links that you need to refer to multiple times across pages here. Delete
any links that you are not going to use.
 -->
</div>
</section></section><section id="aio-07_IGV"><p>Content from <a href="07_IGV.html">IGV</a></p>
<hr>
<p>Last updated on 2025-07-30 |

        <a href="https://github.com/julia-broadbent/intro-to-genome-browsers/edit/main/episodes/07_IGV.md" class="external-link">Edit this page <i aria-hidden="true" data-feather="edit"></i></a></p>
<p>Estimated time: <i aria-hidden="true" data-feather="clock"></i> 11 minutes</p>
<div class="text-end">
          <button role="button" aria-pressed="false" tabindex="0" id="expand-code" class="pull-right" data-expand="Expand All Solutions " data-collapse="Collapse All Solutions "> Expand All Solutions <i aria-hidden="true" data-feather="plus"></i></button>
        </div>
<div class="overview card">
<h2 class="card-header">Overview</h2>
<div class="row g-0">
<div class="col-md-4">
<div class="card-body">
<div class="inner">
<h3 class="card-title">Questions</h3>
<ul>
<li>How do you use IGV?</li>
</ul>
</div>
</div>
</div>
<div class="col-md-8">
<div class="card-body">
<div class="inner bordered">
<h3 class="card-title">Objectives</h3>
<ul>
<li>Learn how to open files and navigate IGV</li>
<li>Understand the type of data that can be visualised in IGV</li>
</ul>
</div>
</div>
</div>
</div>
</div>
<section><h2 class="section-heading" id="igv">IGV<a class="anchor" aria-label="anchor" href="#igv"></a>
</h2>
<hr class="half-width">
<p>In this section we will download a BAM file of gene expression data
from SRA, and view it in the <strong>Integrated Genome Viewer
(IGV)</strong>. BAM files must first be sorted and indexed before they
can be loaded into genome viewers. IGV has tools to do this without
having to use the command line.</p>
</section><section><h2 class="section-heading" id="exercise">Exercise<a class="anchor" aria-label="anchor" href="#exercise"></a>
</h2>
<hr class="half-width">
<div class="section level3">
<h3 id="the-data">The data<a class="anchor" aria-label="anchor" href="#the-data"></a>
</h3>
<p>The expression data we are using for this exercise is from the mouse
<a href="http://casestudies.brain-map.org/celltax" class="external-link">Celltax single cell
expression atlas</a> published by the Allen Brain Institute. The cell
tax vignette has an expression browser that displays gene level
expression as a heat map for any gene of interest. The readsets (fastq
files) and aligned data (BAM files) for 1809 runs on single cells are
also available for download from SRA.</p>
<p>The SRA study ID for this study is <a href="https://www.ncbi.nlm.nih.gov/sra?term=SRP061902" class="external-link">SRP061902</a> and
individual runs from this study are easily selected by viewing the
samples in the ‘RunSelector’.</p>
<p>For this exercise, we will download a few samples in order to
illustrate navigating in IGV by looking at the expression of NTRK2 in
the same cell types we have discussed in earlier exercises. For each
cell type, we will down load a .BAM file containing only the reads from
the chromosome of interest.</p>
</div>
<div class="section level3">
<h3 id="your-task">Your task<a class="anchor" aria-label="anchor" href="#your-task"></a>
</h3>
<div class="section level4">
<h4 id="download-bam-files-from-sra">1. Download BAM files from SRA<a class="anchor" aria-label="anchor" href="#download-bam-files-from-sra"></a>
</h4>
<p>For each SRA run in the table below:</p>
<table class="table">
<colgroup>
<col width="28%">
<col width="38%">
<col width="33%">
</colgroup>
<thead><tr class="header">
<th>Cell type</th>
<th>SRA run</th>
<th>Vignette Cell ID</th>
</tr></thead>
<tbody>
<tr class="odd">
<td>astrocyte</td>
<td><a href="https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?run=SRR2138962" class="external-link">SRR2138962</a></td>
<td>D1319_V</td>
</tr>
<tr class="even">
<td>astrocyte</td>
<td><a href="https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?run=SRR2139935" class="external-link">SRR2139935</a></td>
<td>A1643_VL</td>
</tr>
<tr class="odd">
<td>neuron</td>
<td><a href="https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?run=SRR2139989" class="external-link">SRR2139989</a></td>
<td>S467_V4</td>
</tr>
<tr class="even">
<td>neuron</td>
<td><a href="https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?run=SRR2140047" class="external-link">SRR2140047</a></td>
<td>S1282_V</td>
</tr>
</tbody>
</table>
<ol style="list-style-type: lower-alpha">
<li><p>Click on the SRA run to open the link.</p></li>
<li><p>Click on the ‘Alignment’ tab. Note that the data is aligned to
the mouse GRCm38 genome (mm10).</p></li>
<li><p>Select the chromosome of interest. For NTRK2 in mouse it is
chr13</p></li>
<li><p>For ‘Output this run in:’ select <strong>BAM</strong> and click
on ‘format to:’ <strong>File</strong></p></li>
<li><p>Rename the downloaded file to include the cell type, to avoid
confusion, e.g. <em>SRR2138661_astrocyte_chr13.bam</em></p></li>
</ol>
</div>
<div class="section level4">
<h4 id="use-igv-tools-to-sort-and-index-the-bam-files">2. Use IGV tools to SORT and INDEX the BAM files<a class="anchor" aria-label="anchor" href="#use-igv-tools-to-sort-and-index-the-bam-files"></a>
</h4>
<ol style="list-style-type: lower-alpha">
<li><p>Open IGV and select <code>Tools  &gt;  Run igvtools</code> from
the pull down menus</p></li>
<li><p>Select <code>Sort</code> from the Command options and use the
browse options to select the BAM file you just downloaded. Click
<code>Run</code></p></li>
<li><p>Without closing the igvtools window, now select the command
<code>Index</code> and browse to find the BAM file you just sorted. It
will have the same file name with ‘sorted’ added to the end,
e.g. <em>SRR2138661_astrocyte_chr13.sorted.bam</em></p></li>
</ol>
<p>The resulting index file will have the file name
<em>SRR2138661_astrocyte_chr13.sorted.bam.bai</em></p>
<div id="important" class="callout">
<div class="callout-square">
<i class="callout-icon" data-feather="bell"></i>
</div>
<div id="important" class="callout-inner">
<h3 class="callout-title">IMPORTANT</h3>
<div class="callout-content">
<p>It is essential that the index file for a BAM file has the
<strong>same name</strong> and is located in the <strong>same
folder</strong> as its BAM file. If not, the IVG software will not be
able to open the BAM file.</p>
<figure><img src="../fig/07IGV_BAM_bai_files.png" class="figure mx-auto d-block"></figure>
</div>
</div>
</div>
</div>
<div class="section level4">
<h4 id="view-the-bam-files-in-igv">3. View the BAM files in IGV<a class="anchor" aria-label="anchor" href="#view-the-bam-files-in-igv"></a>
</h4>
<ol style="list-style-type: lower-alpha">
<li><p>Select the Mouse (mm10) genome from the genome box in the top
right hand corner.</p></li>
<li><p>Select <code>File  &gt;  Load from File</code> and select all
four <em>_chr13.sorted.bam</em> files only</p></li>
<li><p>Select <code>open</code> - but don’t expect to see any data yet.
The genome view window opens on a whole chromosome view as default but
it wont show any data until the view region is small enough to show all
data in the current view.</p></li>
<li><p>Type the gene name ‘NTRK2’ into the search window.</p></li>
<li><p>Expand the Refseq gene model track by right clicking it to see
all the splice variants</p></li>
<li>
<p>The gene and thus the genome view is 328kb and the default
setting for viewing data is only 100kb. Unless you have already changed
your settings, alignment data will not get be showing. Zoom into the
region of a coding exon by selecting in the numbered location track at
the top of the genome view.</p>
<p>To see the whole gene in the genome window at the same time you may
need to change the preferences.</p>
</li>
<li><p>Go to <code>View  &gt;  Preferences</code> and select the
<code>Alignments tab</code>. Change the visibility range threshold to
400kb.</p></li>
</ol>
<figure><img src="../fig/07IGV_IGV_preferences.png" alt="You may need to change this back to a smaller range in the future if you are working with large datasets and/or small amounts of memory on your computer" class="figure mx-auto d-block"><div class="figcaption">You may need to change this back to a smaller
range in the future if you are working with large datasets and/or small
amounts of memory on your computer</div>
</figure><figure><img src="../fig/07IGV_Ntrk2.png" class="figure mx-auto d-block"></figure>
</div>
<div class="section level4">
<h4 id="export-images">4. Export images<a class="anchor" aria-label="anchor" href="#export-images"></a>
</h4>
<ol style="list-style-type: lower-alpha">
<li><p>The Genome view above can be exported by selecting
<code>File  &gt;  Save image</code> from the tool bar</p></li>
<li>
<p>To export the Sashimi plot below:</p>
<ol style="list-style-type: lower-roman">
<li>Right click on one of the junction tracks and select
<code>Sashimi Plot</code> from the pull down menu.</li>
<li>Select the tracks you want in your final image.</li>
<li>There are some data filtering and style adjustments you can make to
the Sashimi plot. Right click on each track to access the menu options.
Some changes apply to each track individually and some to all
tracks.</li>
</ol>
</li>
</ol>
<figure><img src="../fig/07IGV_Sashimi.png" class="figure mx-auto d-block"></figure>
</div>
<div class="section level4">
<h4 id="download-and-install-the-gencode-gene-model-annotation-track">5. Download and install the Gencode gene model annotation track<a class="anchor" aria-label="anchor" href="#download-and-install-the-gencode-gene-model-annotation-track"></a>
</h4>
<p>The refseq gene model track is not as comprehensive as GenCode gene
models. For both Human and Mouse the Gencode gene model gtf annotation
files can be downloaded from <a href="https://www.gencodegenes.org/mouse/" class="external-link">Gencode</a>. If you wish to
do this be aware that it takes a little time and is not done as part of
a workshop.</p>
<ol style="list-style-type: lower-alpha">
<li><p>Create a folder called ‘annotations/Mouse’ in the main ‘igv’
folder that was installed on your computer when you downloaded
IGV.</p></li>
<li><p>Download the GTF file from the link above and save it in this
folder.</p></li>
<li><p>Unpack and then <strong>sort</strong> and <strong>index</strong>
the .gtf file using igvtools.</p></li>
<li><p>In IGV, before you load you data files, load this annotation file
and it will replace the refseq one.</p></li>
</ol>
<div id="accordionSpoiler1" class="accordion spoiler-accordion accordion-flush">
<div class="accordion-item">
<button class="accordion-button spoiler-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapseSpoiler1" aria-expanded="false" aria-controls="collapseSpoiler1">
  <h3 class="accordion-header" id="headingSpoiler1">
  <div class="note-square"><i aria-hidden="true" class="callout-icon" data-feather="eye"></i></div>
  FURTHER READING
  </h3>
</button>
<div id="collapseSpoiler1" class="accordion-collapse collapse" aria-labelledby="headingSpoiler1" data-bs-parent="#accordionSpoiler1">
<div class="accordion-body">
<p>If you would like to learn more about how to use IGV, please go
to:</p>
<p><a href="https://rockefelleruniversity.github.io/IGV_course/presentations/singlepage/IGV.html" class="external-link uri">https://rockefelleruniversity.github.io/IGV_course/presentations/singlepage/IGV.html</a></p>
</div>
</div>
</div>
</div>
<div id="keypoints1" class="callout keypoints">
<div class="callout-square">
<i class="callout-icon" data-feather="key"></i>
</div>
<div class="callout-inner">
<h3 class="callout-title">Key Points</h3>
<div class="callout-content">
<ul>
<li>IGV can sort and index BAM files without use of the command
line</li>
<li>Sorted and indexed BAM files can then be opened to view genomic
sequencing and gene expression data in IGV</li>
</ul>
</div>
</div>
</div>
<!--
Place links that you need to refer to multiple times across pages here. Delete
any links that you are not going to use.
 -->
</div>
</div>
</section></section>
</div>
    </main>
</div>
<!-- END  : inst/pkgdown/templates/content-extra.html -->

      </div>
<!--/div.row-->
      		<footer class="row footer mx-md-3"><hr>
<div class="col-md-6">
        <p>This lesson is subject to the <a href="CODE_OF_CONDUCT.html">Code of Conduct</a></p>
        <p>

        <a href="https://github.com/julia-broadbent/intro-to-genome-browsers/edit/main/README.md" class="external-link">Edit on GitHub</a>

	
        | <a href="https://github.com/julia-broadbent/intro-to-genome-browsers/blob/main/CONTRIBUTING.md" class="external-link">Contributing</a>
        | <a href="https://github.com/julia-broadbent/intro-to-genome-browsers/" class="external-link">Source</a></p>
				<p><a href="https://github.com/julia-broadbent/intro-to-genome-browsers/blob/main/CITATION.cff" class="external-link">Cite</a> | <a href="mailto:bioinformatics-training@unimelb.edu.au">Contact</a> | <a href="https://carpentries.org/about/" class="external-link">About</a></p>
			</div>
			<div class="col-md-6">

        <p>Materials licensed under <a href="LICENSE.html">CC-BY 4.0</a> by the authors</p>

        <p>Template licensed under <a href="https://creativecommons.org/licenses/by-sa/4.0/" class="external-link">CC-BY 4.0</a> by <a href="https://carpentries.org/" class="external-link">The Carpentries</a></p>
        <p>Built with <a href="https://github.com/carpentries/sandpaper/tree/0.16.12" class="external-link">sandpaper (0.16.12)</a>, <a href="https://github.com/carpentries/pegboard/tree/0.7.9" class="external-link">pegboard (0.7.9)</a>, and <a href="https://github.com/melbournebioinformatics/uom-varnish/tree/main" class="external-link">varnish (1.0.7.9000)</a></p>
			</div>
		</footer>
</div> <!-- / div.container -->
	<div id="to-top">
		<a href="#top">
      <i class="search-icon" data-feather="arrow-up" role="img" aria-label="Back To Top"></i><br><!-- <span class="d-none d-sm-none d-md-none d-lg-none d-xl-block">Back</span> To Top --><span class="d-none d-sm-none d-md-none d-lg-none d-xl-block">Back</span> To Top
		</a>
	</div>
  <script type="application/ld+json">
    {
  "@context": "https://schema.org",
  "@type": "LearningResource",
  "@id": "melbournebioinformatics.github.io/uom-varnish/instructor/aio.html",
  "inLanguage": "en",
  "dct:conformsTo": "https://bioschemas.org/profiles/LearningResource/1.0-RELEASE",
  "description": "A Carpentries Lesson teaching foundational data and coding skills to researchers worldwide",
  "keywords": "genomics, transcriptomics, IGV, UCSC",
  "name": "All in One View",
  "creativeWorkStatus": "active",
  "url": "melbournebioinformatics.github.io/uom-varnish/instructor/aio.html",
  "identifier": "melbournebioinformatics.github.io/uom-varnish/instructor/aio.html",
  "dateCreated": "2025-07-02",
  "dateModified": "2025-07-30",
  "datePublished": "2025-07-30"
}

  </script><script>
		feather.replace();
	</script>
</body>
</html><!-- END:   inst/pkgdown/templates/layout.html-->

