---
title: Setup
---

## Workshop overview

This workshop will introduce you to the genome browser format and illustrate how 
some freely available genome browsers can be used to interrogate a variety of data types, 
such as gene expression, genomic variation, methylation and many more.

###

::: callout

### Learning objectives

At the end of this introductory workshop, you will:

-   Understand the how some types of genomic and expression data are represented in genome browsers
-   Understand gene models and identify differences between transcript variants
-   Determine the tissue/cell type expression profiles of a gene of interest in mouse and human expression data
-   Know some basic files types used in genome browsers and upload and view local BAM files.
-   Use the BLAT tool to locate genomic regions with similarity to a sequence of interest
-   Create custom interactive views with multiple datatypes to share with colleagues and generate images for publications

:::

By focusing on gene expression, this hands on tutorial will provide beginners with 
an introduction to both the **UCSC Genome Browser** and **IGV (Integrative Genomics Viewer)**. 

Tools and public datasets will be used to illustrate how the expression of transcript 
variants can be investigated in different tissues and cell types using public data.

The data and genes used in this workshop are taken from the neuroscience field, 
however the analysis approaches and tools illustrated can be applied to many research areas.

Public data viewed within the UCSC genome browser in this workshop includes 
human RNAseq data from [GTEX](https://gtexportal.org/home/) and
mouse cell type RNAseq data from [Tabula Muris](https://tabula-muris.ds.czbiohub.org/), while 
a subset of Single cell RNAseq data from 
the [Allen Brain Atlas Celltax](https://celltypes.brain-map.org/) study 
will also be downloaded from [SRA](https://www.ncbi.nlm.nih.gov/sra) and visualised in IGV. 

## Timing

The anticipated workshop duration when delivered to a group of participants is **4 hours**. 
Note that not all the exercises are expected to be completed during the workshop.

## Skill level

This workshop is designed for participants with **no previous experience** using genome browsers 
and **no programming experience**.

## Requirements and preparation

::: checklist
### DEVICE

Attendees are required to provide their own laptop computers
:::

::: checklist
### REQUIRED SOFTWARE

Prior to the workshop:

1.  Download and install [IGV](https://software.broadinstitute.org/software/igv/download) (free)
2.  Ensure that [Chrome](https://www.google.com/chrome/) or [FireFox](https://www.mozilla.org/en-US/) are installed and up to date
3.  Create a user account in the [UCSC Genome Browser](https://genome.ucsc.edu/cgi-bin/hgLogin?hgLogin.do.signupPage=1&returnto=https%3A%2F%2Fgenome.ucsc.edu%2Fcgi%2Dbin%2FhgSession%3Fhgsid%3D2907973544_r0PWRkU4aje0R8U2JQ1u6wewugzj)


This workshop will be run using the above freely available web interfaces and free software with graphical user interfaces. 

:::

::: checklist

### REQUIRED DATA

No additional data needs to be downloaded prior to this workshop. Required data will be downloaded as part of the tutorial exercises.

:::
